declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00015/certs"
declare -x AZ_BATCH_JOB_ID="random_forest"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_2-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00015"
declare -x AZ_BATCH_TASK_ID="task-00015"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00015/wd"
declare -x HOME="/mnt/batch/tasks/workitems/random_forest/job-1/task-00015/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/random_forest/job-1/task-00015/wd"
declare -x PWD="/mnt/batch/tasks/workitems/random_forest/job-1/task-00015/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 21:45:40] Results will not be logged locally.
[2018-04-11 21:45:40] Retrieving tasks...
[2018-04-11 21:45:40] Tasks will be retrieved based on provided task ids.
[2018-04-11 21:45:40] The following tasks were retrieved: [3913].
[2018-04-11 21:45:45] Downloaded task 3913.
[2018-04-11 21:45:46] Prepared pipelines.
[2018-04-11 21:45:46] Started run 1 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 7 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 8 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 2 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 3 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 4 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 5 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:46] Started run 6 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 1: Run already exists in server. Run id(s): {8969779}
[2018-04-11 21:45:50] Error in run 7: Run already exists in server. Run id(s): {9004613}
[2018-04-11 21:45:50] Started run 9 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 8: Run already exists in server. Run id(s): {8969773}
[2018-04-11 21:45:50] Started run 11 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 2: Run already exists in server. Run id(s): {9004607}
[2018-04-11 21:45:50] Error in run 3: Run already exists in server. Run id(s): {8969785}
[2018-04-11 21:45:50] Started run 12 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Started run 10 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Started run 13 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 4: Run already exists in server. Run id(s): {9004609}
[2018-04-11 21:45:50] Started run 14 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 5: Run already exists in server. Run id(s): {9004610}
[2018-04-11 21:45:50] Started run 15 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:50] Error in run 6: Run already exists in server. Run id(s): {8969780, 9017197}
[2018-04-11 21:45:50] Started run 16 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Error in run 11: Run already exists in server. Run id(s): {9004611}
[2018-04-11 21:45:55] Started run 17 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Error in run 9: Run already exists in server. Run id(s): {9004620}
[2018-04-11 21:45:55] Started run 18 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Error in run 15: Run already exists in server. Run id(s): {9004625}
[2018-04-11 21:45:55] Started run 19 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Error in run 12: Run already exists in server. Run id(s): {9004618}
[2018-04-11 21:45:55] Error in run 14: Run already exists in server. Run id(s): {8969784}
[2018-04-11 21:45:55] Error in run 10: Run already exists in server. Run id(s): {8969774}
[2018-04-11 21:45:55] Started run 20 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Error in run 16: Run already exists in server. Run id(s): {9004630}
[2018-04-11 21:45:55] Error in run 13: Run already exists in server. Run id(s): {8969786}
[2018-04-11 21:45:55] Started run 23 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Started run 24 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Started run 21 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:55] Started run 22 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:59] Error in run 17: Run already exists in server. Run id(s): {9004634}
[2018-04-11 21:45:59] Started run 25 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:59] Error in run 24: Run already exists in server. Run id(s): {8969790}
[2018-04-11 21:45:59] Started run 26 on task 3913, dataset 'kc2'.
[2018-04-11 21:45:59] Error in run 21: Run already exists in server. Run id(s): {8969775}
[2018-04-11 21:45:59] Started run 27 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:00] Error in run 22: Run already exists in server. Run id(s): {8969777}
[2018-04-11 21:46:00] Started run 28 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:00] Error in run 18: Run already exists in server. Run id(s): {9004635}
[2018-04-11 21:46:00] Error in run 20: Run already exists in server. Run id(s): {8969782}
[2018-04-11 21:46:00] Error in run 23: Run already exists in server. Run id(s): {9004636}
[2018-04-11 21:46:00] Error in run 19: Run already exists in server. Run id(s): {9004632}
[2018-04-11 21:46:00] Started run 29 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:00] Started run 32 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:00] Started run 30 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:00] Started run 31 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 25: Run already exists in server. Run id(s): {8969783}
[2018-04-11 21:46:04] Started run 33 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 26: Run already exists in server. Run id(s): {8969768}
[2018-04-11 21:46:04] Started run 34 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 27: Run already exists in server. Run id(s): {9004641}
[2018-04-11 21:46:04] Started run 35 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 29: Run already exists in server. Run id(s): {9004649}
[2018-04-11 21:46:04] Error in run 32: Run already exists in server. Run id(s): {9004646}
[2018-04-11 21:46:04] Error in run 28: Run already exists in server. Run id(s): {8969793}
[2018-04-11 21:46:04] Started run 36 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Started run 37 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Started run 38 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 31: Run already exists in server. Run id(s): {8969778}
[2018-04-11 21:46:04] Started run 39 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:04] Error in run 30: Run already exists in server. Run id(s): {8969788}
[2018-04-11 21:46:04] Started run 40 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:08] Error in run 33: Run already exists in server. Run id(s): {9004645}
[2018-04-11 21:46:08] Started run 41 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:08] Error in run 35: Run already exists in server. Run id(s): {9004652}
[2018-04-11 21:46:08] Error in run 34: Run already exists in server. Run id(s): {8969769}
[2018-04-11 21:46:08] Started run 42 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:08] Started run 43 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:09] Error in run 37: Run already exists in server. Run id(s): {9004650}
[2018-04-11 21:46:09] Started run 44 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:09] Error in run 38: Run already exists in server. Run id(s): {9004653}
[2018-04-11 21:46:09] Error in run 36: Run already exists in server. Run id(s): {8969763}
[2018-04-11 21:46:09] Started run 45 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:09] Started run 46 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:09] Error in run 39: Run already exists in server. Run id(s): {8969771}
[2018-04-11 21:46:09] Started run 47 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:09] Error in run 40: Run already exists in server. Run id(s): {9004656}
[2018-04-11 21:46:09] Started run 48 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Error in run 41: Run already exists in server. Run id(s): {9004657}
[2018-04-11 21:46:13] Started run 49 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Error in run 42: Run already exists in server. Run id(s): {9004662}
[2018-04-11 21:46:13] Started run 50 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Error in run 46: Run already exists in server. Run id(s): {8969792}
[2018-04-11 21:46:13] Error in run 44: Run already exists in server. Run id(s): {8969794}
[2018-04-11 21:46:13] Error in run 43: Run already exists in server. Run id(s): {9004661}
[2018-04-11 21:46:13] Error in run 45: Run already exists in server. Run id(s): {9004664}
[2018-04-11 21:46:13] Started run 51 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Started run 52 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Started run 53 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Started run 54 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:13] Error in run 47: Run already exists in server. Run id(s): {9004668}
[2018-04-11 21:46:13] Started run 55 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:14] Error in run 48: Run already exists in server. Run id(s): {8969767}
[2018-04-11 21:46:14] Started run 56 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:17] Error in run 49: Run already exists in server. Run id(s): {8969762}
[2018-04-11 21:46:17] Started run 57 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Error in run 50: Run already exists in server. Run id(s): {8969787}
[2018-04-11 21:46:18] Started run 58 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Error in run 54: Run already exists in server. Run id(s): {9004673}
[2018-04-11 21:46:18] Started run 59 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Error in run 52: Run already exists in server. Run id(s): {8969795}
[2018-04-11 21:46:18] Error in run 55: Run already exists in server. Run id(s): {8969770}
[2018-04-11 21:46:18] Error in run 53: Run already exists in server. Run id(s): {8969764}
[2018-04-11 21:46:18] Error in run 51: Run already exists in server. Run id(s): {9004670}
[2018-04-11 21:46:18] Started run 60 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Started run 62 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Started run 61 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Started run 63 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:18] Error in run 56: Run already exists in server. Run id(s): {9004676}
[2018-04-11 21:46:18] Started run 64 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:22] Error in run 57: Run already exists in server. Run id(s): {9004681}
[2018-04-11 21:46:22] Started run 65 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:22] Error in run 58: Run already exists in server. Run id(s): {8969776}
[2018-04-11 21:46:22] Started run 66 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:22] Error in run 59: Run already exists in server. Run id(s): {9004678}
[2018-04-11 21:46:22] Started run 67 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:23] Error in run 62: Run already exists in server. Run id(s): {8969791}
[2018-04-11 21:46:23] Error in run 61: Run already exists in server. Run id(s): {8969765}
[2018-04-11 21:46:23] Error in run 60: Run already exists in server. Run id(s): {8969781}
[2018-04-11 21:46:23] Started run 68 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:23] Error in run 63: Run already exists in server. Run id(s): {9004680}
[2018-04-11 21:46:23] Error in run 64: Run already exists in server. Run id(s): {8969789}
[2018-04-11 21:46:23] Started run 69 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:23] Started run 72 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:23] Started run 71 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:23] Started run 70 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:26] Error in run 65: Run already exists in server. Run id(s): {8969760}
[2018-04-11 21:46:26] Started run 73 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:26] Error in run 66: Run already exists in server. Run id(s): {9004682}
[2018-04-11 21:46:26] Started run 74 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Error in run 67: Run already exists in server. Run id(s): {8969766}
[2018-04-11 21:46:27] Started run 75 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Error in run 69: Run already exists in server. Run id(s): {9004684}
[2018-04-11 21:46:27] Started run 76 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Error in run 68: Run already exists in server. Run id(s): {8969772}
[2018-04-11 21:46:27] Error in run 72: Run already exists in server. Run id(s): {8969759}
[2018-04-11 21:46:27] Error in run 71: Run already exists in server. Run id(s): {9004688}
[2018-04-11 21:46:27] Started run 77 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Error in run 70: Run already exists in server. Run id(s): {8969761}
[2018-04-11 21:46:27] Started run 78 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Started run 80 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:27] Started run 79 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:31] Error in run 73: Run already exists in server. Run id(s): {9004694}
[2018-04-11 21:46:31] Started run 81 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:31] Error in run 75: Run already exists in server. Run id(s): {9004700}
[2018-04-11 21:46:31] Error in run 74: Run already exists in server. Run id(s): {9004686}
[2018-04-11 21:46:31] Started run 82 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:31] Started run 83 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:32] Error in run 76: Run already exists in server. Run id(s): {9004693}
[2018-04-11 21:46:32] Started run 84 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:32] Error in run 77: Run already exists in server. Run id(s): {9004695}
[2018-04-11 21:46:32] Error in run 78: Run already exists in server. Run id(s): {9004699}
[2018-04-11 21:46:32] Started run 85 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:32] Error in run 80: Run already exists in server. Run id(s): {9004708}
[2018-04-11 21:46:32] Started run 86 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:32] Error in run 79: Run already exists in server. Run id(s): {9004706}
[2018-04-11 21:46:32] Started run 87 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:32] Started run 88 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Error in run 81: Run already exists in server. Run id(s): {9004709}
[2018-04-11 21:46:36] Started run 89 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Error in run 82: Run already exists in server. Run id(s): {9004714}
[2018-04-11 21:46:36] Started run 90 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Error in run 83: Run already exists in server. Run id(s): {9004719}
[2018-04-11 21:46:36] Started run 91 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Error in run 87: Run already exists in server. Run id(s): {9004723}
[2018-04-11 21:46:36] Error in run 84: Run already exists in server. Run id(s): {9004711}
[2018-04-11 21:46:36] Started run 92 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Error in run 85: Run already exists in server. Run id(s): {9004720}
[2018-04-11 21:46:36] Error in run 86: Run already exists in server. Run id(s): {9004718}
[2018-04-11 21:46:36] Error in run 88: Run already exists in server. Run id(s): {9004724}
[2018-04-11 21:46:36] Started run 94 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Started run 95 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Started run 96 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:36] Started run 93 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:40] Error in run 89: Run already exists in server. Run id(s): {9004729}
[2018-04-11 21:46:40] Started run 97 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:40] Error in run 90: Run already exists in server. Run id(s): {9004726}
[2018-04-11 21:46:40] Started run 98 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Error in run 93: Run already exists in server. Run id(s): {9004735}
[2018-04-11 21:46:41] Error in run 95: Run already exists in server. Run id(s): {9004743}
[2018-04-11 21:46:41] Error in run 96: Run already exists in server. Run id(s): {9004737}
[2018-04-11 21:46:41] Started run 99 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Error in run 91: Run already exists in server. Run id(s): {9004728}
[2018-04-11 21:46:41] Error in run 94: Run already exists in server. Run id(s): {9004734}
[2018-04-11 21:46:41] Error in run 92: Run already exists in server. Run id(s): {9004733}
[2018-04-11 21:46:41] Started run 102 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Started run 103 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Started run 100 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Started run 104 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:41] Started run 101 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Error in run 97: Run already exists in server. Run id(s): {9004744}
[2018-04-11 21:46:45] Started run 105 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Error in run 98: Run already exists in server. Run id(s): {9004739}
[2018-04-11 21:46:45] Started run 106 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Error in run 102: Run already exists in server. Run id(s): {9004746}
[2018-04-11 21:46:45] Error in run 99: Run already exists in server. Run id(s): {9004745}
[2018-04-11 21:46:45] Started run 107 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Started run 108 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Error in run 103: Run already exists in server. Run id(s): {9004752}
[2018-04-11 21:46:45] Started run 109 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:45] Error in run 100: Run already exists in server. Run id(s): {9004748}
[2018-04-11 21:46:45] Started run 110 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:46] Error in run 101: Run already exists in server. Run id(s): {9004751}
[2018-04-11 21:46:46] Started run 111 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:46] Error in run 104: Run already exists in server. Run id(s): {9004753}
[2018-04-11 21:46:46] Started run 112 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:49] Error in run 105: Run already exists in server. Run id(s): {9004764}
[2018-04-11 21:46:49] Started run 113 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:49] Error in run 106: Run already exists in server. Run id(s): {9004756}
[2018-04-11 21:46:49] Started run 114 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 107: Run already exists in server. Run id(s): {9004757}
[2018-04-11 21:46:50] Started run 115 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 109: Run already exists in server. Run id(s): {9004763}
[2018-04-11 21:46:50] Started run 116 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 108: Run already exists in server. Run id(s): {9004759}
[2018-04-11 21:46:50] Started run 117 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 110: Run already exists in server. Run id(s): {9004768}
[2018-04-11 21:46:50] Started run 118 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 111: Run already exists in server. Run id(s): {9004766}
[2018-04-11 21:46:50] Started run 119 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:50] Error in run 112: Run already exists in server. Run id(s): {9004765}
[2018-04-11 21:46:50] Started run 120 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 113: Run already exists in server. Run id(s): {9004771}
[2018-04-11 21:46:54] Started run 121 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 114: Run already exists in server. Run id(s): {9004776}
[2018-04-11 21:46:54] Started run 122 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 116: Run already exists in server. Run id(s): {9004784}
[2018-04-11 21:46:54] Started run 123 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 115: Run already exists in server. Run id(s): {9004773}
[2018-04-11 21:46:54] Started run 124 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 117: Run already exists in server. Run id(s): {9004782}
[2018-04-11 21:46:54] Started run 125 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:54] Error in run 118: Run already exists in server. Run id(s): {9004781}
[2018-04-11 21:46:54] Started run 126 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:55] Error in run 119: Run already exists in server. Run id(s): {9004788}
[2018-04-11 21:46:55] Started run 127 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:55] Error in run 120: Run already exists in server. Run id(s): {9004786}
[2018-04-11 21:46:55] Started run 128 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:58] Error in run 121: Run already exists in server. Run id(s): {9004789}
[2018-04-11 21:46:58] Started run 129 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:58] Error in run 122: Run already exists in server. Run id(s): {9004791}
[2018-04-11 21:46:58] Started run 130 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:59] Error in run 126: Run already exists in server. Run id(s): {9004801}
[2018-04-11 21:46:59] Started run 131 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:59] Error in run 127: Run already exists in server. Run id(s): {9004800}
[2018-04-11 21:46:59] Error in run 124: Run already exists in server. Run id(s): {9004798}
[2018-04-11 21:46:59] Error in run 125: Run already exists in server. Run id(s): {9004799}
[2018-04-11 21:46:59] Started run 132 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:59] Error in run 123: Run already exists in server. Run id(s): {9004793}
[2018-04-11 21:46:59] Started run 134 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:59] Started run 135 on task 3913, dataset 'kc2'.
[2018-04-11 21:46:59] Started run 133 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:00] Error in run 128: Run already exists in server. Run id(s): {9004803}
[2018-04-11 21:47:00] Started run 136 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:02] Error in run 129: Run already exists in server. Run id(s): {9004802}
[2018-04-11 21:47:02] Started run 137 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:03] Error in run 130: Run already exists in server. Run id(s): {9004807}
[2018-04-11 21:47:03] Started run 138 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:03] Error in run 131: Run already exists in server. Run id(s): {9004805}
[2018-04-11 21:47:03] Started run 139 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:04] Error in run 133: Run already exists in server. Run id(s): {9004813}
[2018-04-11 21:47:04] Started run 140 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:04] Error in run 135: Run already exists in server. Run id(s): {9004827}
[2018-04-11 21:47:04] Error in run 134: Run already exists in server. Run id(s): {9004821}
[2018-04-11 21:47:04] Error in run 132: Run already exists in server. Run id(s): {9004812}
[2018-04-11 21:47:04] Started run 141 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:04] Started run 142 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:04] Started run 143 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:04] Error in run 136: Run already exists in server. Run id(s): {9004820}
[2018-04-11 21:47:04] Started run 144 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:07] Error in run 137: Run already exists in server. Run id(s): {9004817}
[2018-04-11 21:47:07] Started run 145 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:08] Error in run 138: Run already exists in server. Run id(s): {9004823}
[2018-04-11 21:47:08] Started run 146 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:08] Error in run 139: Run already exists in server. Run id(s): {9004829}
[2018-04-11 21:47:08] Started run 147 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:08] Error in run 140: Run already exists in server. Run id(s): {9004831}
[2018-04-11 21:47:08] Started run 148 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:09] Error in run 143: Run already exists in server. Run id(s): {9004844}
[2018-04-11 21:47:09] Error in run 141: Run already exists in server. Run id(s): {9004834}
[2018-04-11 21:47:09] Started run 149 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:09] Error in run 142: Run already exists in server. Run id(s): {9004837}
[2018-04-11 21:47:09] Started run 150 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:09] Started run 151 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:09] Error in run 144: Run already exists in server. Run id(s): {9004840}
[2018-04-11 21:47:09] Started run 152 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:11] Error in run 145: Run already exists in server. Run id(s): {9004841}
[2018-04-11 21:47:11] Started run 153 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 146: Run already exists in server. Run id(s): {9004842}
[2018-04-11 21:47:13] Started run 154 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 147: Run already exists in server. Run id(s): {9004849}
[2018-04-11 21:47:13] Started run 155 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 148: Run already exists in server. Run id(s): {9004847}
[2018-04-11 21:47:13] Started run 156 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 150: Run already exists in server. Run id(s): {9004853}
[2018-04-11 21:47:13] Started run 157 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 152: Run already exists in server. Run id(s): {9004858}
[2018-04-11 21:47:13] Error in run 149: Run already exists in server. Run id(s): {9004848}
[2018-04-11 21:47:13] Started run 158 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Error in run 151: Run already exists in server. Run id(s): {9004855}
[2018-04-11 21:47:13] Started run 160 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:13] Started run 159 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:16] Error in run 153: Run already exists in server. Run id(s): {9004854}
[2018-04-11 21:47:16] Started run 161 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:17] Error in run 156: Run already exists in server. Run id(s): {9004864}
[2018-04-11 21:47:17] Started run 162 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:17] Error in run 154: Run already exists in server. Run id(s): {9004857}
[2018-04-11 21:47:17] Error in run 155: Run already exists in server. Run id(s): {9004862}
[2018-04-11 21:47:17] Started run 163 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:17] Started run 164 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:18] Error in run 160: Run already exists in server. Run id(s): {9004874}
[2018-04-11 21:47:18] Error in run 157: Run already exists in server. Run id(s): {9004863}
[2018-04-11 21:47:18] Error in run 158: Run already exists in server. Run id(s): {9004867}
[2018-04-11 21:47:18] Error in run 159: Run already exists in server. Run id(s): {9004871}
[2018-04-11 21:47:18] Started run 165 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:18] Started run 168 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:18] Started run 166 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:18] Started run 167 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:21] Error in run 161: Run already exists in server. Run id(s): {9004869}
[2018-04-11 21:47:21] Started run 169 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:22] Error in run 162: Run already exists in server. Run id(s): {9004870}
[2018-04-11 21:47:22] Started run 170 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:22] Error in run 164: Run already exists in server. Run id(s): {9004875}
[2018-04-11 21:47:22] Started run 171 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:22] Error in run 163: Run already exists in server. Run id(s): {9004876}
[2018-04-11 21:47:22] Started run 172 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:23] Error in run 167: Run already exists in server. Run id(s): {9004888}
[2018-04-11 21:47:23] Started run 173 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:23] Error in run 168: Run already exists in server. Run id(s): {9004894}
[2018-04-11 21:47:23] Started run 174 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:23] Error in run 165: Run already exists in server. Run id(s): {9004880}
[2018-04-11 21:47:23] Error in run 166: Run already exists in server. Run id(s): {9004890}
[2018-04-11 21:47:23] Started run 175 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:23] Started run 176 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:25] Error in run 169: Run already exists in server. Run id(s): {9004889}
[2018-04-11 21:47:25] Started run 177 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 170: Run already exists in server. Run id(s): {9004892}
[2018-04-11 21:47:27] Started run 178 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 172: Run already exists in server. Run id(s): {9004895}
[2018-04-11 21:47:27] Started run 179 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 171: Run already exists in server. Run id(s): {9004891}
[2018-04-11 21:47:27] Started run 180 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 173: Run already exists in server. Run id(s): {9004896}
[2018-04-11 21:47:27] Started run 181 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 174: Run already exists in server. Run id(s): {9004898}
[2018-04-11 21:47:27] Started run 182 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Error in run 176: Run already exists in server. Run id(s): {9004901}
[2018-04-11 21:47:27] Error in run 175: Run already exists in server. Run id(s): {9004902}
[2018-04-11 21:47:27] Started run 183 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:27] Started run 184 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:30] Error in run 177: Run already exists in server. Run id(s): {9004909}
[2018-04-11 21:47:30] Started run 185 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:31] Error in run 178: Run already exists in server. Run id(s): {9004904}
[2018-04-11 21:47:31] Started run 186 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:31] Error in run 179: Run already exists in server. Run id(s): {9004911}
[2018-04-11 21:47:31] Started run 187 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:31] Error in run 180: Run already exists in server. Run id(s): {9004908}
[2018-04-11 21:47:31] Started run 188 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:32] Error in run 182: Run already exists in server. Run id(s): {9004926}
[2018-04-11 21:47:32] Started run 189 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:32] Error in run 181: Run already exists in server. Run id(s): {9004912}
[2018-04-11 21:47:32] Started run 190 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:32] Error in run 183: Run already exists in server. Run id(s): {9004918}
[2018-04-11 21:47:32] Started run 191 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:32] Error in run 184: Run already exists in server. Run id(s): {9004921}
[2018-04-11 21:47:32] Started run 192 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:34] Error in run 185: Run already exists in server. Run id(s): {9004922}
[2018-04-11 21:47:34] Started run 193 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Error in run 186: Run already exists in server. Run id(s): {9004927}
[2018-04-11 21:47:36] Started run 194 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Error in run 187: Run already exists in server. Run id(s): {9004930}
[2018-04-11 21:47:36] Error in run 188: Run already exists in server. Run id(s): {9004924}
[2018-04-11 21:47:36] Started run 195 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Started run 196 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Error in run 189: Run already exists in server. Run id(s): {9004929}
[2018-04-11 21:47:36] Error in run 190: Run already exists in server. Run id(s): {9004928}
[2018-04-11 21:47:36] Started run 197 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Started run 198 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Error in run 192: Run already exists in server. Run id(s): {9004933}
[2018-04-11 21:47:36] Error in run 191: Run already exists in server. Run id(s): {9004932}
[2018-04-11 21:47:36] Started run 199 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:36] Started run 200 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:39] Error in run 193: Run already exists in server. Run id(s): {9004942}
[2018-04-11 21:47:39] Started run 201 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:40] Error in run 195: Run already exists in server. Run id(s): {9004943}
[2018-04-11 21:47:40] Started run 202 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:40] Error in run 194: Run already exists in server. Run id(s): {9004938}
[2018-04-11 21:47:40] Error in run 196: Run already exists in server. Run id(s): {9004940}
[2018-04-11 21:47:40] Started run 203 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:40] Started run 204 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:41] Error in run 198: Run already exists in server. Run id(s): {9004946}
[2018-04-11 21:47:41] Error in run 200: Run already exists in server. Run id(s): {9004951}
[2018-04-11 21:47:41] Started run 205 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:41] Error in run 197: Run already exists in server. Run id(s): {9004947}
[2018-04-11 21:47:41] Error in run 199: Run already exists in server. Run id(s): {9004948}
[2018-04-11 21:47:41] Started run 207 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:41] Started run 208 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:41] Started run 206 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:44] Error in run 201: Run already exists in server. Run id(s): {9004953}
[2018-04-11 21:47:44] Started run 209 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:45] Error in run 202: Run already exists in server. Run id(s): {9004957}
[2018-04-11 21:47:45] Started run 210 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:45] Error in run 203: Run already exists in server. Run id(s): {9004962}
[2018-04-11 21:47:45] Started run 211 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:45] Error in run 204: Run already exists in server. Run id(s): {9004960}
[2018-04-11 21:47:45] Started run 212 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:46] Error in run 205: Run already exists in server. Run id(s): {9004963}
[2018-04-11 21:47:46] Error in run 206: Run already exists in server. Run id(s): {9004964}
[2018-04-11 21:47:46] Error in run 207: Run already exists in server. Run id(s): {9004966}
[2018-04-11 21:47:46] Started run 213 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:46] Error in run 208: Run already exists in server. Run id(s): {9004973}
[2018-04-11 21:47:46] Started run 216 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:46] Started run 214 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:46] Started run 215 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:48] Error in run 209: Run already exists in server. Run id(s): {9004971}
[2018-04-11 21:47:48] Started run 217 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:49] Error in run 210: Run already exists in server. Run id(s): {9004974}
[2018-04-11 21:47:49] Started run 218 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:49] Error in run 211: Run already exists in server. Run id(s): {9004975}
[2018-04-11 21:47:49] Started run 219 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:50] Error in run 212: Run already exists in server. Run id(s): {9004982}
[2018-04-11 21:47:50] Started run 220 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:50] Error in run 213: Run already exists in server. Run id(s): {9004985}
[2018-04-11 21:47:50] Started run 221 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:51] Error in run 214: Run already exists in server. Run id(s): {8969802}
[2018-04-11 21:47:51] Error in run 215: Run already exists in server. Run id(s): {9004983}
[2018-04-11 21:47:51] Started run 222 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:51] Started run 223 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:51] Error in run 216: Run already exists in server. Run id(s): {9004986}
[2018-04-11 21:47:51] Started run 224 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:53] Error in run 217: Run already exists in server. Run id(s): {9004990}
[2018-04-11 21:47:53] Started run 225 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:54] Error in run 218: Run already exists in server. Run id(s): {9004991}
[2018-04-11 21:47:54] Started run 226 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:54] Error in run 219: Run already exists in server. Run id(s): {9004995}
[2018-04-11 21:47:54] Started run 227 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:54] Error in run 220: Run already exists in server. Run id(s): {9004993}
[2018-04-11 21:47:54] Started run 228 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:55] Error in run 221: Run already exists in server. Run id(s): {9004998}
[2018-04-11 21:47:55] Started run 229 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:55] Error in run 222: Run already exists in server. Run id(s): {9004996}
[2018-04-11 21:47:55] Error in run 223: Run already exists in server. Run id(s): {9004997}
[2018-04-11 21:47:55] Started run 230 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:55] Started run 231 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:55] Error in run 224: Run already exists in server. Run id(s): {9005002}
[2018-04-11 21:47:55] Started run 232 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:58] Error in run 225: Run already exists in server. Run id(s): {9005005}
[2018-04-11 21:47:58] Started run 233 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:58] Error in run 226: Run already exists in server. Run id(s): {9005003}
[2018-04-11 21:47:58] Started run 234 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:58] Error in run 227: Run already exists in server. Run id(s): {9005007}
[2018-04-11 21:47:58] Started run 235 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:59] Error in run 228: Run already exists in server. Run id(s): {9005013}
[2018-04-11 21:47:59] Started run 236 on task 3913, dataset 'kc2'.
[2018-04-11 21:47:59] Error in run 229: Run already exists in server. Run id(s): {9005019}
[2018-04-11 21:47:59] Started run 237 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:00] Error in run 231: Run already exists in server. Run id(s): {9005014}
[2018-04-11 21:48:00] Error in run 230: Run already exists in server. Run id(s): {9005015}
[2018-04-11 21:48:00] Started run 238 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:00] Started run 239 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:00] Error in run 232: Run already exists in server. Run id(s): {9005023}
[2018-04-11 21:48:00] Started run 240 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:02] Error in run 233: Run already exists in server. Run id(s): {9005018}
[2018-04-11 21:48:02] Started run 241 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:03] Error in run 234: Run already exists in server. Run id(s): {9005022}
[2018-04-11 21:48:03] Started run 242 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:03] Error in run 235: Run already exists in server. Run id(s): {9005029}
[2018-04-11 21:48:03] Started run 243 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:03] Error in run 236: Run already exists in server. Run id(s): {9005030}
[2018-04-11 21:48:03] Started run 244 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:04] Error in run 237: Run already exists in server. Run id(s): {8969803}
[2018-04-11 21:48:04] Started run 245 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:04] Error in run 239: Run already exists in server. Run id(s): {9005037}
[2018-04-11 21:48:04] Error in run 238: Run already exists in server. Run id(s): {9005032}
[2018-04-11 21:48:04] Started run 246 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:04] Started run 247 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:04] Error in run 240: Run already exists in server. Run id(s): {9005036}
[2018-04-11 21:48:04] Started run 248 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:07] Error in run 241: Run already exists in server. Run id(s): {9005035}
[2018-04-11 21:48:07] Started run 249 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:07] Error in run 242: Run already exists in server. Run id(s): {9005040}
[2018-04-11 21:48:07] Started run 250 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:08] Error in run 243: Run already exists in server. Run id(s): {9005045}
[2018-04-11 21:48:08] Started run 251 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:08] Error in run 244: Run already exists in server. Run id(s): {9005044}
[2018-04-11 21:48:08] Started run 252 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:09] Error in run 248: Run already exists in server. Run id(s): {9005050}
[2018-04-11 21:48:09] Error in run 247: Run already exists in server. Run id(s): {9005051}
[2018-04-11 21:48:09] Error in run 245: Run already exists in server. Run id(s): {9005046}
[2018-04-11 21:48:09] Started run 253 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:09] Error in run 246: Run already exists in server. Run id(s): {9005047}
[2018-04-11 21:48:09] Started run 254 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:09] Started run 256 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:09] Started run 255 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:11] Error in run 249: Run already exists in server. Run id(s): {9005048}
[2018-04-11 21:48:11] Started run 257 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:12] Error in run 251: Run already exists in server. Run id(s): {9005060}
[2018-04-11 21:48:12] Error in run 250: Run already exists in server. Run id(s): {9005052}
[2018-04-11 21:48:12] Started run 258 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:12] Started run 259 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:13] Error in run 252: Run already exists in server. Run id(s): {9005058}
[2018-04-11 21:48:13] Started run 260 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:13] Error in run 253: Run already exists in server. Run id(s): {9005063}
[2018-04-11 21:48:13] Started run 261 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:14] Error in run 254: Run already exists in server. Run id(s): {9005062}
[2018-04-11 21:48:14] Error in run 255: Run already exists in server. Run id(s): {9005061}
[2018-04-11 21:48:14] Error in run 256: Run already exists in server. Run id(s): {9005067}
[2018-04-11 21:48:14] Started run 262 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:14] Started run 263 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:14] Started run 264 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:16] Error in run 257: Run already exists in server. Run id(s): {9005066}
[2018-04-11 21:48:16] Started run 265 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:17] Error in run 259: Run already exists in server. Run id(s): {9005076}
[2018-04-11 21:48:17] Error in run 258: Run already exists in server. Run id(s): {9005081}
[2018-04-11 21:48:17] Started run 266 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:17] Started run 267 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:17] Error in run 260: Run already exists in server. Run id(s): {8969813}
[2018-04-11 21:48:17] Started run 268 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:18] Error in run 261: Run already exists in server. Run id(s): {9005078}
[2018-04-11 21:48:18] Started run 269 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:18] Error in run 263: Run already exists in server. Run id(s): {9005079}
[2018-04-11 21:48:18] Error in run 262: Run already exists in server. Run id(s): {9017929, 9005077}
[2018-04-11 21:48:18] Started run 270 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:18] Started run 271 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:18] Error in run 264: Run already exists in server. Run id(s): {8969809}
[2018-04-11 21:48:18] Started run 272 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:20] Error in run 265: Run already exists in server. Run id(s): {9005082}
[2018-04-11 21:48:20] Started run 273 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:21] Error in run 266: Run already exists in server. Run id(s): {9005080}
[2018-04-11 21:48:21] Started run 274 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:21] Error in run 267: Run already exists in server. Run id(s): {9005086}
[2018-04-11 21:48:21] Started run 275 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:22] Error in run 268: Run already exists in server. Run id(s): {9005085}
[2018-04-11 21:48:22] Started run 276 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:22] Error in run 269: Run already exists in server. Run id(s): {8969805}
[2018-04-11 21:48:22] Started run 277 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:23] Error in run 270: Run already exists in server. Run id(s): {9005091}
[2018-04-11 21:48:23] Error in run 271: Run already exists in server. Run id(s): {9005087}
[2018-04-11 21:48:23] Started run 278 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:23] Started run 279 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:23] Error in run 272: Run already exists in server. Run id(s): {8969801}
[2018-04-11 21:48:23] Started run 280 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:25] Error in run 273: Run already exists in server. Run id(s): {9005088}
[2018-04-11 21:48:25] Started run 281 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:26] Error in run 274: Run already exists in server. Run id(s): {8969806}
[2018-04-11 21:48:26] Started run 282 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:26] Error in run 275: Run already exists in server. Run id(s): {9005099}
[2018-04-11 21:48:26] Started run 283 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:26] Error in run 276: Run already exists in server. Run id(s): {9005090}
[2018-04-11 21:48:26] Started run 284 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:27] Error in run 277: Run already exists in server. Run id(s): {9005094}
[2018-04-11 21:48:27] Started run 285 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:27] Error in run 279: Run already exists in server. Run id(s): {9005101}
[2018-04-11 21:48:27] Error in run 278: Run already exists in server. Run id(s): {9005097}
[2018-04-11 21:48:27] Started run 286 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:27] Started run 287 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:27] Error in run 280: Run already exists in server. Run id(s): {9005103}
[2018-04-11 21:48:27] Started run 288 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:29] Error in run 281: Run already exists in server. Run id(s): {8969800}
[2018-04-11 21:48:29] Started run 289 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:30] Error in run 283: Run already exists in server. Run id(s): {9005108}
[2018-04-11 21:48:30] Error in run 282: Run already exists in server. Run id(s): {9005104}
[2018-04-11 21:48:30] Started run 290 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:30] Started run 291 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:31] Error in run 284: Run already exists in server. Run id(s): {9005106}
[2018-04-11 21:48:31] Started run 292 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:32] Error in run 285: Run already exists in server. Run id(s): {9005115}
[2018-04-11 21:48:32] Started run 293 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:32] Error in run 286: Run already exists in server. Run id(s): {9005112}
[2018-04-11 21:48:32] Started run 294 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:32] Error in run 287: Run already exists in server. Run id(s): {9005113}
[2018-04-11 21:48:32] Started run 295 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:32] Error in run 288: Run already exists in server. Run id(s): {9005116}
[2018-04-11 21:48:32] Started run 296 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:34] Error in run 289: Run already exists in server. Run id(s): {8969815}
[2018-04-11 21:48:34] Started run 297 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:35] Error in run 291: Run already exists in server. Run id(s): {9005120}
[2018-04-11 21:48:35] Error in run 290: Run already exists in server. Run id(s): {8969817}
[2018-04-11 21:48:35] Started run 298 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:35] Started run 299 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:35] Error in run 292: Run already exists in server. Run id(s): {9005117}
[2018-04-11 21:48:35] Started run 300 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:36] Error in run 293: Run already exists in server. Run id(s): {9005127}
[2018-04-11 21:48:36] Started run 301 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:36] Error in run 294: Run already exists in server. Run id(s): {9005122}
[2018-04-11 21:48:36] Started run 302 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:37] Error in run 295: Run already exists in server. Run id(s): {8969814}
[2018-04-11 21:48:37] Started run 303 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:37] Error in run 296: Run already exists in server. Run id(s): {9005128}
[2018-04-11 21:48:37] Started run 304 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:38] Error in run 297: Run already exists in server. Run id(s): {9005126}
[2018-04-11 21:48:38] Started run 305 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:39] Error in run 299: Run already exists in server. Run id(s): {9005133}
[2018-04-11 21:48:39] Error in run 298: Run already exists in server. Run id(s): {9005130}
[2018-04-11 21:48:39] Started run 306 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:39] Started run 307 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:40] Error in run 300: Run already exists in server. Run id(s): {9005139}
[2018-04-11 21:48:40] Started run 308 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:41] Error in run 301: Run already exists in server. Run id(s): {9005140}
[2018-04-11 21:48:41] Started run 309 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:41] Error in run 302: Run already exists in server. Run id(s): {9005148}
[2018-04-11 21:48:41] Started run 310 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:41] Error in run 304: Run already exists in server. Run id(s): {9005142}
[2018-04-11 21:48:41] Error in run 303: Run already exists in server. Run id(s): {8969812}
[2018-04-11 21:48:41] Started run 311 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:41] Started run 312 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:43] Error in run 305: Run already exists in server. Run id(s): {9005150}
[2018-04-11 21:48:43] Started run 313 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:44] Error in run 306: Run already exists in server. Run id(s): {8969804}
[2018-04-11 21:48:44] Error in run 307: Run already exists in server. Run id(s): {8969807}
[2018-04-11 21:48:44] Started run 314 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:44] Started run 315 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:45] Error in run 308: Run already exists in server. Run id(s): {9005153}
[2018-04-11 21:48:45] Started run 316 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:45] Error in run 310: Run already exists in server. Run id(s): {9005146}
[2018-04-11 21:48:45] Started run 317 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:45] Error in run 309: Run already exists in server. Run id(s): {8969818}
[2018-04-11 21:48:45] Started run 318 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:46] Error in run 312: Run already exists in server. Run id(s): {8969810}
[2018-04-11 21:48:46] Error in run 311: Run already exists in server. Run id(s): {9005149}
[2018-04-11 21:48:46] Started run 319 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:46] Started run 320 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:48] Error in run 313: Run already exists in server. Run id(s): {8969808}
[2018-04-11 21:48:48] Started run 321 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:48] Error in run 315: Run already exists in server. Run id(s): {9005157}
[2018-04-11 21:48:48] Started run 322 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:48] Error in run 314: Run already exists in server. Run id(s): {9005159}
[2018-04-11 21:48:48] Started run 323 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:49] Error in run 316: Run already exists in server. Run id(s): {8969816}
[2018-04-11 21:48:49] Started run 324 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:50] Error in run 317: Run already exists in server. Run id(s): {8969826}
[2018-04-11 21:48:50] Error in run 318: Run already exists in server. Run id(s): {8969811}
[2018-04-11 21:48:50] Started run 325 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:50] Started run 326 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:50] Error in run 320: Run already exists in server. Run id(s): {9005162}
[2018-04-11 21:48:50] Error in run 319: Run already exists in server. Run id(s): {9005164}
[2018-04-11 21:48:50] Started run 327 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:50] Started run 328 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:52] Error in run 321: Run already exists in server. Run id(s): {8969823}
[2018-04-11 21:48:52] Started run 329 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:53] Error in run 322: Run already exists in server. Run id(s): {9005169}
[2018-04-11 21:48:53] Started run 330 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:53] Error in run 323: Run already exists in server. Run id(s): {8969819}
[2018-04-11 21:48:53] Started run 331 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:53] Error in run 324: Run already exists in server. Run id(s): {9005171}
[2018-04-11 21:48:53] Started run 332 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:54] Error in run 325: Run already exists in server. Run id(s): {8969842}
[2018-04-11 21:48:54] Started run 333 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:54] Error in run 326: Run already exists in server. Run id(s): {9005170}
[2018-04-11 21:48:54] Started run 334 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:55] Error in run 328: Run already exists in server. Run id(s): {9005178}
[2018-04-11 21:48:55] Error in run 327: Run already exists in server. Run id(s): {8969820}
[2018-04-11 21:48:55] Started run 335 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:55] Started run 336 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:56] Error in run 329: Run already exists in server. Run id(s): {9005187}
[2018-04-11 21:48:56] Started run 337 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:57] Error in run 330: Run already exists in server. Run id(s): {9005175}
[2018-04-11 21:48:57] Started run 338 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:58] Error in run 331: Run already exists in server. Run id(s): {9005188}
[2018-04-11 21:48:58] Started run 339 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:58] Error in run 332: Run already exists in server. Run id(s): {8969835}
[2018-04-11 21:48:58] Started run 340 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:59] Error in run 333: Run already exists in server. Run id(s): {8969828}
[2018-04-11 21:48:59] Error in run 334: Run already exists in server. Run id(s): {9005193}
[2018-04-11 21:48:59] Started run 341 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:59] Started run 342 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:59] Error in run 335: Run already exists in server. Run id(s): {9005197}
[2018-04-11 21:48:59] Started run 343 on task 3913, dataset 'kc2'.
[2018-04-11 21:48:59] Error in run 336: Run already exists in server. Run id(s): {8969836}
[2018-04-11 21:48:59] Started run 344 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:01] Error in run 337: Run already exists in server. Run id(s): {9005194}
[2018-04-11 21:49:01] Started run 345 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:02] Error in run 338: Run already exists in server. Run id(s): {8969829}
[2018-04-11 21:49:02] Started run 346 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:02] Error in run 339: Run already exists in server. Run id(s): {9005198}
[2018-04-11 21:49:02] Started run 347 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:03] Error in run 340: Run already exists in server. Run id(s): {8969824}
[2018-04-11 21:49:03] Started run 348 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:03] Error in run 341: Run already exists in server. Run id(s): {9005202}
[2018-04-11 21:49:03] Started run 349 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:04] Error in run 342: Run already exists in server. Run id(s): {9005203}
[2018-04-11 21:49:04] Started run 350 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:04] Error in run 343: Run already exists in server. Run id(s): {9005210}
[2018-04-11 21:49:04] Started run 351 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:04] Error in run 344: Run already exists in server. Run id(s): {9005208}
[2018-04-11 21:49:04] Started run 352 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:06] Error in run 345: Run already exists in server. Run id(s): {8969821}
[2018-04-11 21:49:06] Started run 353 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:07] Error in run 346: Run already exists in server. Run id(s): {8969832}
[2018-04-11 21:49:07] Error in run 347: Run already exists in server. Run id(s): {8969837}
[2018-04-11 21:49:07] Started run 354 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:07] Started run 355 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:08] Error in run 348: Run already exists in server. Run id(s): {9005211}
[2018-04-11 21:49:08] Started run 356 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:08] Error in run 349: Run already exists in server. Run id(s): {9005215}
[2018-04-11 21:49:08] Started run 357 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:08] Error in run 352: Run already exists in server. Run id(s): {8969830}
[2018-04-11 21:49:08] Started run 358 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:08] Error in run 351: Run already exists in server. Run id(s): {9005213}
[2018-04-11 21:49:08] Error in run 350: Run already exists in server. Run id(s): {9005218}
[2018-04-11 21:49:08] Started run 359 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:08] Started run 360 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:10] Error in run 353: Run already exists in server. Run id(s): {8969825}
[2018-04-11 21:49:10] Started run 361 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:11] Error in run 354: Run already exists in server. Run id(s): {9005220}
[2018-04-11 21:49:11] Started run 362 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:11] Error in run 355: Run already exists in server. Run id(s): {9005221}
[2018-04-11 21:49:11] Started run 363 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:12] Error in run 356: Run already exists in server. Run id(s): {8969838}
[2018-04-11 21:49:12] Error in run 357: Run already exists in server. Run id(s): {9005228}
[2018-04-11 21:49:12] Started run 364 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:12] Started run 365 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:13] Error in run 358: Run already exists in server. Run id(s): {8969833}
[2018-04-11 21:49:13] Error in run 359: Run already exists in server. Run id(s): {8969831}
[2018-04-11 21:49:13] Error in run 360: Run already exists in server. Run id(s): {9005230}
[2018-04-11 21:49:13] Started run 366 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:13] Started run 367 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:13] Started run 368 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:15] Error in run 361: Run already exists in server. Run id(s): {9005235}
[2018-04-11 21:49:15] Started run 369 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:16] Error in run 362: Run already exists in server. Run id(s): {9005232}
[2018-04-11 21:49:16] Started run 370 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:16] Error in run 363: Run already exists in server. Run id(s): {9005237}
[2018-04-11 21:49:16] Started run 371 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:17] Error in run 364: Run already exists in server. Run id(s): {9005238}
[2018-04-11 21:49:17] Started run 372 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:17] Error in run 365: Run already exists in server. Run id(s): {9005234}
[2018-04-11 21:49:17] Started run 373 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:18] Error in run 366: Run already exists in server. Run id(s): {9005242}
[2018-04-11 21:49:18] Started run 374 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:18] Error in run 368: Run already exists in server. Run id(s): {9005244}
[2018-04-11 21:49:18] Error in run 367: Run already exists in server. Run id(s): {9005248}
[2018-04-11 21:49:18] Started run 375 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:18] Started run 376 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:19] Error in run 369: Run already exists in server. Run id(s): {9005249}
[2018-04-11 21:49:19] Started run 377 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:20] Error in run 370: Run already exists in server. Run id(s): {8969840}
[2018-04-11 21:49:20] Started run 378 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:21] Error in run 371: Run already exists in server. Run id(s): {8969839}
[2018-04-11 21:49:21] Started run 379 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:22] Error in run 372: Run already exists in server. Run id(s): {9005250}
[2018-04-11 21:49:22] Started run 380 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:22] Error in run 373: Run already exists in server. Run id(s): {9005253}
[2018-04-11 21:49:22] Started run 381 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:22] Error in run 374: Run already exists in server. Run id(s): {9005251}
[2018-04-11 21:49:22] Started run 382 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:22] Error in run 376: Run already exists in server. Run id(s): {9005255}
[2018-04-11 21:49:22] Started run 383 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:23] Error in run 375: Run already exists in server. Run id(s): {8969841}
[2018-04-11 21:49:23] Started run 384 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:24] Error in run 377: Run already exists in server. Run id(s): {9005260}
[2018-04-11 21:49:24] Started run 385 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:25] Error in run 378: Run already exists in server. Run id(s): {9005266}
[2018-04-11 21:49:25] Started run 386 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:25] Error in run 379: Run already exists in server. Run id(s): {9005259}
[2018-04-11 21:49:25] Started run 387 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:26] Error in run 380: Run already exists in server. Run id(s): {9005271}
[2018-04-11 21:49:26] Started run 388 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:26] Error in run 381: Run already exists in server. Run id(s): {9005267}
[2018-04-11 21:49:26] Started run 389 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:27] Error in run 382: Run already exists in server. Run id(s): {9005268}
[2018-04-11 21:49:27] Started run 390 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:27] Error in run 383: Run already exists in server. Run id(s): {9005272}
[2018-04-11 21:49:27] Started run 391 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:27] Error in run 384: Run already exists in server. Run id(s): {9005269}
[2018-04-11 21:49:27] Started run 392 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:28] Error in run 385: Run already exists in server. Run id(s): {9005275}
[2018-04-11 21:49:28] Started run 393 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:30] Error in run 386: Run already exists in server. Run id(s): {9005274}
[2018-04-11 21:49:30] Started run 394 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:30] Error in run 387: Run already exists in server. Run id(s): {9005278}
[2018-04-11 21:49:30] Started run 395 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:31] Error in run 388: Run already exists in server. Run id(s): {9005286}
[2018-04-11 21:49:31] Started run 396 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:31] Error in run 389: Run already exists in server. Run id(s): {9005298}
[2018-04-11 21:49:31] Started run 397 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:31] Error in run 390: Run already exists in server. Run id(s): {9005292}
[2018-04-11 21:49:31] Started run 398 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:31] Error in run 391: Run already exists in server. Run id(s): {9005287}
[2018-04-11 21:49:31] Started run 399 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:32] Error in run 392: Run already exists in server. Run id(s): {9005288}
[2018-04-11 21:49:32] Started run 400 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:33] Error in run 393: Run already exists in server. Run id(s): {9005294}
[2018-04-11 21:49:33] Started run 401 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:34] Error in run 394: Run already exists in server. Run id(s): {9005297}
[2018-04-11 21:49:34] Started run 402 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:34] Error in run 395: Run already exists in server. Run id(s): {9005295}
[2018-04-11 21:49:34] Started run 403 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:35] Error in run 396: Run already exists in server. Run id(s): {9005305}
[2018-04-11 21:49:35] Started run 404 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:35] Error in run 397: Run already exists in server. Run id(s): {9005303}
[2018-04-11 21:49:35] Started run 405 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:36] Error in run 398: Run already exists in server. Run id(s): {9005300}
[2018-04-11 21:49:36] Started run 406 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:36] Error in run 399: Run already exists in server. Run id(s): {9005306}
[2018-04-11 21:49:36] Started run 407 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:36] Error in run 400: Run already exists in server. Run id(s): {9005314}
[2018-04-11 21:49:36] Started run 408 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:38] Error in run 401: Run already exists in server. Run id(s): {9005307}
[2018-04-11 21:49:38] Started run 409 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:39] Error in run 402: Run already exists in server. Run id(s): {9005311}
[2018-04-11 21:49:39] Started run 410 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:39] Error in run 403: Run already exists in server. Run id(s): {9005310}
[2018-04-11 21:49:39] Started run 411 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:40] Error in run 404: Run already exists in server. Run id(s): {9005316}
[2018-04-11 21:49:40] Started run 412 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:40] Error in run 405: Run already exists in server. Run id(s): {9005318}
[2018-04-11 21:49:40] Started run 413 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:40] Error in run 406: Run already exists in server. Run id(s): {9005322}
[2018-04-11 21:49:40] Started run 414 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:41] Error in run 407: Run already exists in server. Run id(s): {9005319}
[2018-04-11 21:49:41] Started run 415 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:41] Error in run 408: Run already exists in server. Run id(s): {9005328}
[2018-04-11 21:49:41] Started run 416 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:42] Error in run 409: Run already exists in server. Run id(s): {9005326}
[2018-04-11 21:49:42] Started run 417 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:43] Error in run 410: Run already exists in server. Run id(s): {9005329}
[2018-04-11 21:49:43] Started run 418 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:43] Error in run 411: Run already exists in server. Run id(s): {9005331}
[2018-04-11 21:49:43] Started run 419 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:44] Error in run 412: Run already exists in server. Run id(s): {9005325}
[2018-04-11 21:49:44] Started run 420 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:45] Error in run 413: Run already exists in server. Run id(s): {9005327}
[2018-04-11 21:49:45] Started run 421 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:45] Error in run 414: Run already exists in server. Run id(s): {9005332}
[2018-04-11 21:49:45] Started run 422 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:45] Error in run 415: Run already exists in server. Run id(s): {9005334}
[2018-04-11 21:49:45] Started run 423 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:46] Error in run 416: Run already exists in server. Run id(s): {9005340}
[2018-04-11 21:49:46] Started run 424 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:47] Error in run 417: Run already exists in server. Run id(s): {9005356}
[2018-04-11 21:49:47] Started run 425 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:48] Error in run 419: Run already exists in server. Run id(s): {9005347}
[2018-04-11 21:49:48] Error in run 418: Run already exists in server. Run id(s): {9005342}
[2018-04-11 21:49:48] Started run 426 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:48] Started run 427 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:49] Error in run 420: Run already exists in server. Run id(s): {9005345}
[2018-04-11 21:49:49] Started run 428 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:49] Error in run 421: Run already exists in server. Run id(s): {9005348}
[2018-04-11 21:49:49] Started run 429 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:49] Error in run 422: Run already exists in server. Run id(s): {9005346}
[2018-04-11 21:49:49] Started run 430 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:49] Error in run 423: Run already exists in server. Run id(s): {9005353}
[2018-04-11 21:49:49] Started run 431 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:50] Error in run 424: Run already exists in server. Run id(s): {9005357}
[2018-04-11 21:49:50] Started run 432 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:52] Error in run 425: Run already exists in server. Run id(s): {9005359}
[2018-04-11 21:49:52] Started run 433 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:52] Error in run 426: Run already exists in server. Run id(s): {9005363}
[2018-04-11 21:49:52] Error in run 427: Run already exists in server. Run id(s): {9005361}
[2018-04-11 21:49:52] Started run 434 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:52] Started run 435 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:54] Error in run 428: Run already exists in server. Run id(s): {9005365}
[2018-04-11 21:49:54] Started run 436 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:54] Error in run 429: Run already exists in server. Run id(s): {9005366}
[2018-04-11 21:49:54] Started run 437 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:54] Error in run 430: Run already exists in server. Run id(s): {9005370}
[2018-04-11 21:49:54] Started run 438 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:54] Error in run 431: Run already exists in server. Run id(s): {9005376}
[2018-04-11 21:49:54] Started run 439 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:55] Error in run 432: Run already exists in server. Run id(s): {9005371}
[2018-04-11 21:49:55] Started run 440 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:57] Error in run 433: Run already exists in server. Run id(s): {9005375}
[2018-04-11 21:49:57] Started run 441 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:57] Error in run 434: Run already exists in server. Run id(s): {9005381}
[2018-04-11 21:49:57] Started run 442 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:57] Error in run 435: Run already exists in server. Run id(s): {9005382}
[2018-04-11 21:49:57] Started run 443 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:58] Error in run 436: Run already exists in server. Run id(s): {9005380}
[2018-04-11 21:49:58] Started run 444 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:58] Error in run 437: Run already exists in server. Run id(s): {9005383}
[2018-04-11 21:49:58] Started run 445 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:58] Error in run 438: Run already exists in server. Run id(s): {9005387}
[2018-04-11 21:49:58] Started run 446 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:59] Error in run 439: Run already exists in server. Run id(s): {9005394}
[2018-04-11 21:49:59] Started run 447 on task 3913, dataset 'kc2'.
[2018-04-11 21:49:59] Error in run 440: Run already exists in server. Run id(s): {9005388}
[2018-04-11 21:49:59] Started run 448 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:01] Error in run 441: Run already exists in server. Run id(s): {9005391}
[2018-04-11 21:50:01] Started run 449 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:01] Error in run 442: Run already exists in server. Run id(s): {9005392}
[2018-04-11 21:50:01] Started run 450 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:01] Error in run 443: Run already exists in server. Run id(s): {9005401}
[2018-04-11 21:50:01] Started run 451 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:03] Error in run 444: Run already exists in server. Run id(s): {9005393}
[2018-04-11 21:50:03] Started run 452 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:03] Error in run 445: Run already exists in server. Run id(s): {9005396}
[2018-04-11 21:50:03] Started run 453 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:03] Error in run 446: Run already exists in server. Run id(s): {9005407}
[2018-04-11 21:50:03] Started run 454 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:03] Error in run 447: Run already exists in server. Run id(s): {9005410}
[2018-04-11 21:50:03] Started run 455 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:04] Error in run 448: Run already exists in server. Run id(s): {9005415}
[2018-04-11 21:50:04] Started run 456 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:06] Error in run 449: Run already exists in server. Run id(s): {9005409}
[2018-04-11 21:50:06] Started run 457 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:06] Error in run 450: Run already exists in server. Run id(s): {9005416}
[2018-04-11 21:50:06] Error in run 451: Run already exists in server. Run id(s): {9005408}
[2018-04-11 21:50:06] Started run 458 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:06] Started run 459 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:07] Error in run 452: Run already exists in server. Run id(s): {9005420}
[2018-04-11 21:50:07] Started run 460 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:08] Error in run 455: Run already exists in server. Run id(s): {9005428}
[2018-04-11 21:50:08] Started run 461 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:08] Error in run 454: Run already exists in server. Run id(s): {9005422}
[2018-04-11 21:50:08] Started run 462 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:08] Error in run 453: Run already exists in server. Run id(s): {9005412}
[2018-04-11 21:50:08] Started run 463 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:08] Error in run 456: Run already exists in server. Run id(s): {9005426}
[2018-04-11 21:50:08] Started run 464 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:10] Error in run 457: Run already exists in server. Run id(s): {9005429}
[2018-04-11 21:50:10] Started run 465 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:11] Error in run 458: Run already exists in server. Run id(s): {9005442}
[2018-04-11 21:50:11] Error in run 459: Run already exists in server. Run id(s): {9005435}
[2018-04-11 21:50:11] Started run 466 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:11] Started run 467 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:12] Error in run 460: Run already exists in server. Run id(s): {9005433}
[2018-04-11 21:50:12] Started run 468 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:12] Error in run 461: Run already exists in server. Run id(s): {9005440}
[2018-04-11 21:50:12] Started run 469 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:13] Error in run 463: Run already exists in server. Run id(s): {9005445}
[2018-04-11 21:50:13] Error in run 462: Run already exists in server. Run id(s): {9005439}
[2018-04-11 21:50:13] Started run 470 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:13] Error in run 464: Run already exists in server. Run id(s): {9005448}
[2018-04-11 21:50:13] Started run 471 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:13] Started run 472 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:15] Error in run 465: Run already exists in server. Run id(s): {9005447}
[2018-04-11 21:50:15] Started run 473 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:16] Error in run 466: Run already exists in server. Run id(s): {9005453}
[2018-04-11 21:50:16] Error in run 467: Run already exists in server. Run id(s): {9005459}
[2018-04-11 21:50:16] Started run 474 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:16] Started run 475 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:17] Error in run 468: Run already exists in server. Run id(s): {9005454}
[2018-04-11 21:50:17] Started run 476 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:17] Error in run 469: Run already exists in server. Run id(s): {9005456}
[2018-04-11 21:50:17] Started run 477 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:17] Error in run 471: Run already exists in server. Run id(s): {9005460}
[2018-04-11 21:50:17] Started run 478 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:17] Error in run 470: Run already exists in server. Run id(s): {9005457}
[2018-04-11 21:50:17] Started run 479 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:17] Error in run 472: Run already exists in server. Run id(s): {9005467}
[2018-04-11 21:50:17] Started run 480 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:20] Error in run 473: Run already exists in server. Run id(s): {9005462}
[2018-04-11 21:50:20] Started run 481 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:20] Error in run 474: Run already exists in server. Run id(s): {9005477}
[2018-04-11 21:50:20] Started run 482 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:20] Error in run 475: Run already exists in server. Run id(s): {9005475}
[2018-04-11 21:50:20] Started run 483 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:21] Error in run 476: Run already exists in server. Run id(s): {9005471}
[2018-04-11 21:50:21] Started run 484 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:21] Error in run 477: Run already exists in server. Run id(s): {9005470}
[2018-04-11 21:50:21] Started run 485 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:22] Error in run 478: Run already exists in server. Run id(s): {9005478}
[2018-04-11 21:50:22] Started run 486 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:22] Error in run 480: Run already exists in server. Run id(s): {9005480}
[2018-04-11 21:50:22] Error in run 479: Run already exists in server. Run id(s): {9005476}
[2018-04-11 21:50:22] Started run 487 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:22] Started run 488 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:24] Error in run 481: Run already exists in server. Run id(s): {9005488}
[2018-04-11 21:50:24] Started run 489 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:25] Error in run 482: Run already exists in server. Run id(s): {9005486}
[2018-04-11 21:50:25] Started run 490 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:25] Error in run 483: Run already exists in server. Run id(s): {9005490}
[2018-04-11 21:50:25] Started run 491 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:26] Error in run 484: Run already exists in server. Run id(s): {9005495}
[2018-04-11 21:50:26] Error in run 485: Run already exists in server. Run id(s): {9005493}
[2018-04-11 21:50:26] Started run 492 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:26] Started run 493 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:26] Error in run 486: Run already exists in server. Run id(s): {9005498}
[2018-04-11 21:50:26] Started run 494 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:27] Error in run 487: Run already exists in server. Run id(s): {9005499}
[2018-04-11 21:50:27] Error in run 488: Run already exists in server. Run id(s): {9005497}
[2018-04-11 21:50:27] Started run 495 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:27] Started run 496 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:29] Error in run 489: Run already exists in server. Run id(s): {9005505}
[2018-04-11 21:50:29] Started run 497 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:29] Error in run 490: Run already exists in server. Run id(s): {9005508}
[2018-04-11 21:50:29] Started run 498 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:30] Error in run 491: Run already exists in server. Run id(s): {9005509}
[2018-04-11 21:50:30] Started run 499 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:31] Error in run 492: Run already exists in server. Run id(s): {9005516}
[2018-04-11 21:50:31] Error in run 493: Run already exists in server. Run id(s): {9005517}
[2018-04-11 21:50:31] Started run 500 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:31] Started run 501 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:31] Error in run 494: Run already exists in server. Run id(s): {9005514, 9018901}
[2018-04-11 21:50:31] Started run 502 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:32] Error in run 496: Run already exists in server. Run id(s): {9005524}
[2018-04-11 21:50:32] Started run 503 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:32] Error in run 495: Run already exists in server. Run id(s): {9005522}
[2018-04-11 21:50:32] Started run 504 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:33] Error in run 497: Run already exists in server. Run id(s): {9005530}
[2018-04-11 21:50:33] Started run 505 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:34] Error in run 498: Run already exists in server. Run id(s): {9005527}
[2018-04-11 21:50:34] Started run 506 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:35] Error in run 499: Run already exists in server. Run id(s): {9005523}
[2018-04-11 21:50:35] Started run 507 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:36] Error in run 502: Run already exists in server. Run id(s): {9005531}
[2018-04-11 21:50:36] Started run 508 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:36] Error in run 500: Run already exists in server. Run id(s): {9005528}
[2018-04-11 21:50:36] Error in run 501: Run already exists in server. Run id(s): {9005535}
[2018-04-11 21:50:36] Started run 509 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:36] Started run 510 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:36] Error in run 503: Run already exists in server. Run id(s): {9005540}
[2018-04-11 21:50:36] Started run 511 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:36] Error in run 504: Run already exists in server. Run id(s): {9005542}
[2018-04-11 21:50:36] Started run 512 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:38] Error in run 505: Run already exists in server. Run id(s): {9005544}
[2018-04-11 21:50:38] Started run 513 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:39] Error in run 506: Run already exists in server. Run id(s): {9005546}
[2018-04-11 21:50:39] Started run 514 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:39] Error in run 507: Run already exists in server. Run id(s): {9005541}
[2018-04-11 21:50:39] Started run 515 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:40] Error in run 508: Run already exists in server. Run id(s): {9005545}
[2018-04-11 21:50:40] Started run 516 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:40] Error in run 509: Run already exists in server. Run id(s): {9005552}
[2018-04-11 21:50:40] Error in run 510: Run already exists in server. Run id(s): {9005550}
[2018-04-11 21:50:40] Started run 517 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:40] Started run 518 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:41] Error in run 511: Run already exists in server. Run id(s): {9005558}
[2018-04-11 21:50:41] Error in run 512: Run already exists in server. Run id(s): {9005565}
[2018-04-11 21:50:41] Started run 519 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:41] Started run 520 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:43] Error in run 513: Run already exists in server. Run id(s): {9005559}
[2018-04-11 21:50:43] Started run 521 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:43] Error in run 514: Run already exists in server. Run id(s): {9019045, 9005566}
[2018-04-11 21:50:43] Started run 522 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:44] Error in run 515: Run already exists in server. Run id(s): {9005567}
[2018-04-11 21:50:44] Started run 523 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:45] Error in run 516: Run already exists in server. Run id(s): {9005568}
[2018-04-11 21:50:45] Started run 524 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:45] Error in run 518: Run already exists in server. Run id(s): {9005569}
[2018-04-11 21:50:45] Error in run 517: Run already exists in server. Run id(s): {9005572}
[2018-04-11 21:50:45] Started run 525 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:45] Started run 526 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:46] Error in run 520: Run already exists in server. Run id(s): {9005574}
[2018-04-11 21:50:46] Started run 527 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:46] Error in run 519: Run already exists in server. Run id(s): {9005582}
[2018-04-11 21:50:46] Started run 528 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:48] Error in run 521: Run already exists in server. Run id(s): {9005577}
[2018-04-11 21:50:48] Started run 529 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:48] Error in run 522: Run already exists in server. Run id(s): {9005575}
[2018-04-11 21:50:48] Started run 530 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:49] Error in run 523: Run already exists in server. Run id(s): {9005586}
[2018-04-11 21:50:49] Started run 531 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:49] Error in run 524: Run already exists in server. Run id(s): {9005578}
[2018-04-11 21:50:49] Started run 532 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:50] Error in run 526: Run already exists in server. Run id(s): {9005588}
[2018-04-11 21:50:50] Started run 533 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:50] Error in run 525: Run already exists in server. Run id(s): {9005579}
[2018-04-11 21:50:50] Started run 534 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:51] Error in run 527: Run already exists in server. Run id(s): {9005599}
[2018-04-11 21:50:51] Error in run 528: Run already exists in server. Run id(s): {9005594}
[2018-04-11 21:50:51] Started run 535 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:51] Started run 536 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:52] Error in run 529: Run already exists in server. Run id(s): {9005596}
[2018-04-11 21:50:52] Started run 537 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:52] Error in run 530: Run already exists in server. Run id(s): {9005595}
[2018-04-11 21:50:52] Started run 538 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:54] Error in run 531: Run already exists in server. Run id(s): {9005592}
[2018-04-11 21:50:54] Error in run 532: Run already exists in server. Run id(s): {9005598}
[2018-04-11 21:50:54] Started run 539 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:54] Started run 540 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:55] Error in run 534: Run already exists in server. Run id(s): {9005601}
[2018-04-11 21:50:55] Error in run 533: Run already exists in server. Run id(s): {9005597}
[2018-04-11 21:50:55] Started run 541 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:55] Started run 542 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:55] Error in run 535: Run already exists in server. Run id(s): {9005605}
[2018-04-11 21:50:55] Error in run 536: Run already exists in server. Run id(s): {9005608}
[2018-04-11 21:50:55] Started run 543 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:55] Started run 544 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:57] Error in run 538: Run already exists in server. Run id(s): {9005616}
[2018-04-11 21:50:57] Started run 545 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:57] Error in run 537: Run already exists in server. Run id(s): {9005612}
[2018-04-11 21:50:57] Started run 546 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:59] Error in run 539: Run already exists in server. Run id(s): {9005614}
[2018-04-11 21:50:59] Started run 547 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:59] Error in run 540: Run already exists in server. Run id(s): {9005613}
[2018-04-11 21:50:59] Started run 548 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:59] Error in run 541: Run already exists in server. Run id(s): {9005619}
[2018-04-11 21:50:59] Started run 549 on task 3913, dataset 'kc2'.
[2018-04-11 21:50:59] Error in run 542: Run already exists in server. Run id(s): {9005620}
[2018-04-11 21:50:59] Started run 550 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:00] Error in run 543: Run already exists in server. Run id(s): {9005626}
[2018-04-11 21:51:00] Error in run 544: Run already exists in server. Run id(s): {9005627}
[2018-04-11 21:51:00] Started run 551 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:00] Started run 552 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:02] Error in run 545: Run already exists in server. Run id(s): {9005628}
[2018-04-11 21:51:02] Started run 553 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:02] Error in run 546: Run already exists in server. Run id(s): {9005630}
[2018-04-11 21:51:02] Started run 554 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:03] Error in run 547: Run already exists in server. Run id(s): {9005635}
[2018-04-11 21:51:03] Error in run 548: Run already exists in server. Run id(s): {9005636}
[2018-04-11 21:51:03] Started run 555 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:03] Started run 556 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:04] Error in run 550: Run already exists in server. Run id(s): {9005639}
[2018-04-11 21:51:04] Error in run 549: Run already exists in server. Run id(s): {9005633}
[2018-04-11 21:51:04] Started run 557 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:04] Started run 558 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:05] Error in run 552: Run already exists in server. Run id(s): {9005642}
[2018-04-11 21:51:05] Error in run 551: Run already exists in server. Run id(s): {9005641}
[2018-04-11 21:51:05] Started run 559 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:05] Started run 560 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:06] Error in run 553: Run already exists in server. Run id(s): {9005645}
[2018-04-11 21:51:06] Started run 561 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:07] Error in run 554: Run already exists in server. Run id(s): {9005646}
[2018-04-11 21:51:07] Started run 562 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:08] Error in run 556: Run already exists in server. Run id(s): {9005651}
[2018-04-11 21:51:08] Error in run 555: Run already exists in server. Run id(s): {9005655}
[2018-04-11 21:51:08] Started run 563 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:08] Started run 564 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:09] Error in run 558: Run already exists in server. Run id(s): {9005653}
[2018-04-11 21:51:09] Error in run 557: Run already exists in server. Run id(s): {9005649}
[2018-04-11 21:51:09] Started run 565 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:09] Started run 566 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:10] Error in run 560: Run already exists in server. Run id(s): {9005658, 9019309}
[2018-04-11 21:51:10] Started run 567 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:10] Error in run 559: Run already exists in server. Run id(s): {9005660}
[2018-04-11 21:51:10] Started run 568 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:11] Error in run 561: Run already exists in server. Run id(s): {9005664}
[2018-04-11 21:51:11] Started run 569 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:11] Error in run 562: Run already exists in server. Run id(s): {9005661}
[2018-04-11 21:51:11] Started run 570 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:13] Error in run 564: Run already exists in server. Run id(s): {9005666}
[2018-04-11 21:51:13] Error in run 563: Run already exists in server. Run id(s): {9005665}
[2018-04-11 21:51:13] Started run 571 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:13] Started run 572 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:14] Error in run 565: Run already exists in server. Run id(s): {9005667}
[2018-04-11 21:51:14] Error in run 566: Run already exists in server. Run id(s): {9005672}
[2018-04-11 21:51:14] Started run 573 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:14] Started run 574 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:14] Error in run 567: Run already exists in server. Run id(s): {9005671}
[2018-04-11 21:51:14] Started run 575 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:14] Error in run 568: Run already exists in server. Run id(s): {9005677}
[2018-04-11 21:51:14] Started run 576 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:15] Error in run 569: Run already exists in server. Run id(s): {9005676}
[2018-04-11 21:51:15] Started run 577 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:16] Error in run 570: Run already exists in server. Run id(s): {9005675}
[2018-04-11 21:51:16] Started run 578 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:18] Error in run 571: Run already exists in server. Run id(s): {9005694}
[2018-04-11 21:51:18] Started run 579 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:18] Error in run 572: Run already exists in server. Run id(s): {9005681}
[2018-04-11 21:51:18] Started run 580 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:18] Error in run 573: Run already exists in server. Run id(s): {9005680}
[2018-04-11 21:51:18] Started run 581 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:18] Error in run 574: Run already exists in server. Run id(s): {9005689}
[2018-04-11 21:51:18] Started run 582 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:19] Error in run 575: Run already exists in server. Run id(s): {9005686}
[2018-04-11 21:51:19] Started run 583 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:19] Error in run 576: Run already exists in server. Run id(s): {8969857}
[2018-04-11 21:51:19] Started run 584 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:20] Error in run 577: Run already exists in server. Run id(s): {9005688}
[2018-04-11 21:51:20] Started run 585 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:21] Error in run 578: Run already exists in server. Run id(s): {9005691}
[2018-04-11 21:51:21] Started run 586 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:22] Error in run 580: Run already exists in server. Run id(s): {9005705}
[2018-04-11 21:51:22] Started run 587 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:22] Error in run 579: Run already exists in server. Run id(s): {9005695}
[2018-04-11 21:51:22] Started run 588 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:23] Error in run 581: Run already exists in server. Run id(s): {9005702}
[2018-04-11 21:51:23] Started run 589 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:23] Error in run 583: Run already exists in server. Run id(s): {8969881}
[2018-04-11 21:51:23] Error in run 582: Run already exists in server. Run id(s): {8969882}
[2018-04-11 21:51:23] Started run 590 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:23] Started run 591 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:24] Error in run 584: Run already exists in server. Run id(s): {9005708}
[2018-04-11 21:51:24] Started run 592 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:25] Error in run 585: Run already exists in server. Run id(s): {9005715}
[2018-04-11 21:51:25] Started run 593 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:25] Error in run 586: Run already exists in server. Run id(s): {9005711}
[2018-04-11 21:51:25] Started run 594 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:27] Error in run 587: Run already exists in server. Run id(s): {9005710}
[2018-04-11 21:51:27] Started run 595 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:27] Error in run 588: Run already exists in server. Run id(s): {9005709}
[2018-04-11 21:51:27] Started run 596 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:27] Error in run 589: Run already exists in server. Run id(s): {9005714}
[2018-04-11 21:51:27] Started run 597 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:28] Error in run 591: Run already exists in server. Run id(s): {9005718}
[2018-04-11 21:51:28] Started run 598 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:28] Error in run 590: Run already exists in server. Run id(s): {9005717}
[2018-04-11 21:51:28] Started run 599 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:29] Error in run 592: Run already exists in server. Run id(s): {9005723}
[2018-04-11 21:51:29] Started run 600 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:29] Error in run 593: Run already exists in server. Run id(s): {9005729}
[2018-04-11 21:51:29] Started run 601 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:30] Error in run 594: Run already exists in server. Run id(s): {9019377, 9005726}
[2018-04-11 21:51:30] Started run 602 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:32] Error in run 595: Run already exists in server. Run id(s): {9005727}
[2018-04-11 21:51:32] Started run 603 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:32] Error in run 597: Run already exists in server. Run id(s): {9005728}
[2018-04-11 21:51:32] Error in run 596: Run already exists in server. Run id(s): {8969850}
[2018-04-11 21:51:32] Started run 604 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:32] Started run 605 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:33] Error in run 598: Run already exists in server. Run id(s): {9005730}
[2018-04-11 21:51:33] Error in run 599: Run already exists in server. Run id(s): {9005732}
[2018-04-11 21:51:33] Started run 606 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:33] Started run 607 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:33] Error in run 600: Run already exists in server. Run id(s): {9005738}
[2018-04-11 21:51:33] Started run 608 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:34] Error in run 601: Run already exists in server. Run id(s): {9005740}
[2018-04-11 21:51:34] Started run 609 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:34] Error in run 602: Run already exists in server. Run id(s): {9005737}
[2018-04-11 21:51:34] Started run 610 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:36] Error in run 603: Run already exists in server. Run id(s): {8969856}
[2018-04-11 21:51:36] Started run 611 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:37] Error in run 605: Run already exists in server. Run id(s): {8969865}
[2018-04-11 21:51:37] Error in run 604: Run already exists in server. Run id(s): {9005749}
[2018-04-11 21:51:37] Started run 612 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:37] Started run 613 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:37] Error in run 606: Run already exists in server. Run id(s): {8969853}
[2018-04-11 21:51:37] Started run 614 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:37] Error in run 607: Run already exists in server. Run id(s): {9005747}
[2018-04-11 21:51:37] Started run 615 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:38] Error in run 608: Run already exists in server. Run id(s): {9005750}
[2018-04-11 21:51:38] Started run 616 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:38] Error in run 609: Run already exists in server. Run id(s): {9005752}
[2018-04-11 21:51:38] Started run 617 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:39] Error in run 610: Run already exists in server. Run id(s): {8969847}
[2018-04-11 21:51:39] Started run 618 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:41] Error in run 611: Run already exists in server. Run id(s): {8969854}
[2018-04-11 21:51:41] Started run 619 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:41] Error in run 613: Run already exists in server. Run id(s): {8969852}
[2018-04-11 21:51:41] Error in run 612: Run already exists in server. Run id(s): {9005756}
[2018-04-11 21:51:41] Started run 620 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:41] Started run 621 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:42] Error in run 614: Run already exists in server. Run id(s): {9005757}
[2018-04-11 21:51:42] Started run 622 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:42] Error in run 615: Run already exists in server. Run id(s): {9005755}
[2018-04-11 21:51:42] Started run 623 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:42] Error in run 616: Run already exists in server. Run id(s): {9005762}
[2018-04-11 21:51:42] Started run 624 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:43] Error in run 617: Run already exists in server. Run id(s): {8969849}
[2018-04-11 21:51:43] Started run 625 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:44] Error in run 618: Run already exists in server. Run id(s): {8969863}
[2018-04-11 21:51:44] Started run 626 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:45] Error in run 619: Run already exists in server. Run id(s): {9005761}
[2018-04-11 21:51:45] Started run 627 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:46] Error in run 621: Run already exists in server. Run id(s): {9005767}
[2018-04-11 21:51:46] Error in run 620: Run already exists in server. Run id(s): {9005764}
[2018-04-11 21:51:46] Started run 628 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:46] Error in run 622: Run already exists in server. Run id(s): {8969851}
[2018-04-11 21:51:46] Started run 629 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:46] Started run 630 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:47] Error in run 623: Run already exists in server. Run id(s): {9005766}
[2018-04-11 21:51:47] Started run 631 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:47] Error in run 624: Run already exists in server. Run id(s): {9005772}
[2018-04-11 21:51:47] Started run 632 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:48] Error in run 625: Run already exists in server. Run id(s): {9005769}
[2018-04-11 21:51:48] Started run 633 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:48] Error in run 626: Run already exists in server. Run id(s): {8969862}
[2018-04-11 21:51:48] Started run 634 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:50] Error in run 627: Run already exists in server. Run id(s): {9005775}
[2018-04-11 21:51:50] Started run 635 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:51] Error in run 629: Run already exists in server. Run id(s): {9005774}
[2018-04-11 21:51:51] Error in run 630: Run already exists in server. Run id(s): {9005780}
[2018-04-11 21:51:51] Started run 636 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:51] Error in run 628: Run already exists in server. Run id(s): {9005779}
[2018-04-11 21:51:51] Started run 638 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:51] Started run 637 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:51] Error in run 631: Run already exists in server. Run id(s): {8969868}
[2018-04-11 21:51:51] Started run 639 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:52] Error in run 632: Run already exists in server. Run id(s): {9005786}
[2018-04-11 21:51:52] Started run 640 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:52] Error in run 633: Run already exists in server. Run id(s): {9005785}
[2018-04-11 21:51:52] Started run 641 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:52] Error in run 634: Run already exists in server. Run id(s): {8969879}
[2018-04-11 21:51:52] Started run 642 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:55] Error in run 635: Run already exists in server. Run id(s): {8969861}
[2018-04-11 21:51:55] Started run 643 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:56] Error in run 636: Run already exists in server. Run id(s): {9005794}
[2018-04-11 21:51:56] Error in run 638: Run already exists in server. Run id(s): {8969864}
[2018-04-11 21:51:56] Started run 644 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:56] Error in run 637: Run already exists in server. Run id(s): {8969855}
[2018-04-11 21:51:56] Started run 646 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:56] Started run 645 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:56] Error in run 639: Run already exists in server. Run id(s): {9005795}
[2018-04-11 21:51:56] Started run 647 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:56] Error in run 640: Run already exists in server. Run id(s): {8969875}
[2018-04-11 21:51:56] Started run 648 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:57] Error in run 642: Run already exists in server. Run id(s): {9005797}
[2018-04-11 21:51:57] Error in run 641: Run already exists in server. Run id(s): {9005800}
[2018-04-11 21:51:57] Started run 649 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:57] Started run 650 on task 3913, dataset 'kc2'.
[2018-04-11 21:51:59] Error in run 643: Run already exists in server. Run id(s): {8969867}
[2018-04-11 21:51:59] Started run 651 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:00] Error in run 646: Run already exists in server. Run id(s): {8969873}
[2018-04-11 21:52:00] Error in run 644: Run already exists in server. Run id(s): {8969876}
[2018-04-11 21:52:00] Started run 652 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:00] Started run 653 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:00] Error in run 647: Run already exists in server. Run id(s): {9005806}
[2018-04-11 21:52:00] Error in run 645: Run already exists in server. Run id(s): {9005796}
[2018-04-11 21:52:00] Started run 654 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:00] Started run 655 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:01] Error in run 648: Run already exists in server. Run id(s): {8969871}
[2018-04-11 21:52:01] Started run 656 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:02] Error in run 649: Run already exists in server. Run id(s): {9005805}
[2018-04-11 21:52:02] Started run 657 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:02] Error in run 650: Run already exists in server. Run id(s): {8969869}
[2018-04-11 21:52:02] Started run 658 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:04] Error in run 651: Run already exists in server. Run id(s): {9005811}
[2018-04-11 21:52:04] Started run 659 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:05] Error in run 652: Run already exists in server. Run id(s): {8969877}
[2018-04-11 21:52:05] Started run 660 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:05] Error in run 653: Run already exists in server. Run id(s): {8969858}
[2018-04-11 21:52:05] Started run 661 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:05] Error in run 655: Run already exists in server. Run id(s): {9005816}
[2018-04-11 21:52:05] Error in run 654: Run already exists in server. Run id(s): {8969888}
[2018-04-11 21:52:05] Started run 662 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:05] Started run 663 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:05] Error in run 656: Run already exists in server. Run id(s): {8969878}
[2018-04-11 21:52:05] Started run 664 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:07] Error in run 657: Run already exists in server. Run id(s): {8969880}
[2018-04-11 21:52:07] Error in run 658: Run already exists in server. Run id(s): {9005814}
[2018-04-11 21:52:07] Started run 665 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:07] Started run 666 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:08] Error in run 659: Run already exists in server. Run id(s): {8969874}
[2018-04-11 21:52:08] Started run 667 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:09] Error in run 660: Run already exists in server. Run id(s): {8969883}
[2018-04-11 21:52:09] Started run 668 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:09] Error in run 661: Run already exists in server. Run id(s): {9005820}
[2018-04-11 21:52:09] Started run 669 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:10] Error in run 662: Run already exists in server. Run id(s): {8969859}
[2018-04-11 21:52:10] Started run 670 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:10] Error in run 663: Run already exists in server. Run id(s): {8969860}
[2018-04-11 21:52:10] Started run 671 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:10] Error in run 664: Run already exists in server. Run id(s): {9005819}
[2018-04-11 21:52:10] Started run 672 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:11] Error in run 665: Run already exists in server. Run id(s): {8969866}
[2018-04-11 21:52:11] Error in run 666: Run already exists in server. Run id(s): {9005825}
[2018-04-11 21:52:11] Started run 673 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:11] Started run 674 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:13] Error in run 667: Run already exists in server. Run id(s): {9005823}
[2018-04-11 21:52:13] Started run 675 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:14] Error in run 668: Run already exists in server. Run id(s): {9005821}
[2018-04-11 21:52:14] Started run 676 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:14] Error in run 669: Run already exists in server. Run id(s): {8969885}
[2018-04-11 21:52:14] Started run 677 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:14] Error in run 670: Run already exists in server. Run id(s): {8969870}
[2018-04-11 21:52:14] Started run 678 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:14] Error in run 671: Run already exists in server. Run id(s): {9005830}
[2018-04-11 21:52:14] Started run 679 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:15] Error in run 672: Run already exists in server. Run id(s): {9005833}
[2018-04-11 21:52:15] Started run 680 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:16] Error in run 674: Run already exists in server. Run id(s): {9005837}
[2018-04-11 21:52:16] Error in run 673: Run already exists in server. Run id(s): {9005834}
[2018-04-11 21:52:16] Started run 681 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:16] Started run 682 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:17] Error in run 675: Run already exists in server. Run id(s): {9005836}
[2018-04-11 21:52:17] Started run 683 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:18] Error in run 676: Run already exists in server. Run id(s): {9005841}
[2018-04-11 21:52:18] Started run 684 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:19] Error in run 677: Run already exists in server. Run id(s): {9005838}
[2018-04-11 21:52:19] Started run 685 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:19] Error in run 678: Run already exists in server. Run id(s): {9005840}
[2018-04-11 21:52:19] Started run 686 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:19] Error in run 679: Run already exists in server. Run id(s): {8969884}
[2018-04-11 21:52:19] Started run 687 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:20] Error in run 680: Run already exists in server. Run id(s): {9005849}
[2018-04-11 21:52:20] Started run 688 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:21] Error in run 681: Run already exists in server. Run id(s): {9005846}
[2018-04-11 21:52:21] Started run 689 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:21] Error in run 682: Run already exists in server. Run id(s): {9005850}
[2018-04-11 21:52:21] Started run 690 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:22] Error in run 683: Run already exists in server. Run id(s): {9005848}
[2018-04-11 21:52:22] Started run 691 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:23] Error in run 684: Run already exists in server. Run id(s): {9005854}
[2018-04-11 21:52:23] Started run 692 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:23] Error in run 685: Run already exists in server. Run id(s): {8969886}
[2018-04-11 21:52:23] Error in run 686: Run already exists in server. Run id(s): {9005856}
[2018-04-11 21:52:23] Started run 693 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:23] Started run 694 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:23] Error in run 687: Run already exists in server. Run id(s): {9005857}
[2018-04-11 21:52:23] Started run 695 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:24] Error in run 688: Run already exists in server. Run id(s): {9005858}
[2018-04-11 21:52:24] Started run 696 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:25] Error in run 689: Run already exists in server. Run id(s): {9005860}
[2018-04-11 21:52:25] Started run 697 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:26] Error in run 690: Run already exists in server. Run id(s): {9005868}
[2018-04-11 21:52:26] Started run 698 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:27] Error in run 691: Run already exists in server. Run id(s): {9005863}
[2018-04-11 21:52:27] Started run 699 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:27] Error in run 692: Run already exists in server. Run id(s): {9005865}
[2018-04-11 21:52:27] Started run 700 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:28] Error in run 694: Run already exists in server. Run id(s): {9005880}
[2018-04-11 21:52:28] Error in run 693: Run already exists in server. Run id(s): {9005872}
[2018-04-11 21:52:28] Started run 701 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:28] Started run 702 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:28] Error in run 695: Run already exists in server. Run id(s): {9005875}
[2018-04-11 21:52:28] Started run 703 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:29] Error in run 696: Run already exists in server. Run id(s): {9005879}
[2018-04-11 21:52:29] Started run 704 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:30] Error in run 697: Run already exists in server. Run id(s): {9005881}
[2018-04-11 21:52:30] Started run 705 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:31] Error in run 698: Run already exists in server. Run id(s): {9005884}
[2018-04-11 21:52:31] Started run 706 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:31] Error in run 699: Run already exists in server. Run id(s): {9005883}
[2018-04-11 21:52:31] Started run 707 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:32] Error in run 700: Run already exists in server. Run id(s): {9005885}
[2018-04-11 21:52:32] Started run 708 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:32] Error in run 701: Run already exists in server. Run id(s): {9005887}
[2018-04-11 21:52:32] Started run 709 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:33] Error in run 702: Run already exists in server. Run id(s): {9005888}
[2018-04-11 21:52:33] Started run 710 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:33] Error in run 703: Run already exists in server. Run id(s): {9005893}
[2018-04-11 21:52:33] Started run 711 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:33] Error in run 704: Run already exists in server. Run id(s): {9005890}
[2018-04-11 21:52:33] Started run 712 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:34] Error in run 705: Run already exists in server. Run id(s): {9005894}
[2018-04-11 21:52:34] Started run 713 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:35] Error in run 706: Run already exists in server. Run id(s): {9005906}
[2018-04-11 21:52:35] Started run 714 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:36] Error in run 707: Run already exists in server. Run id(s): {9005904}
[2018-04-11 21:52:36] Started run 715 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:37] Error in run 708: Run already exists in server. Run id(s): {9005900}
[2018-04-11 21:52:37] Started run 716 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:37] Error in run 709: Run already exists in server. Run id(s): {9005903}
[2018-04-11 21:52:37] Started run 717 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:37] Error in run 710: Run already exists in server. Run id(s): {9005901}
[2018-04-11 21:52:37] Started run 718 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:38] Error in run 711: Run already exists in server. Run id(s): {9005908}
[2018-04-11 21:52:38] Started run 719 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:38] Error in run 712: Run already exists in server. Run id(s): {9005909}
[2018-04-11 21:52:38] Started run 720 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:39] Error in run 713: Run already exists in server. Run id(s): {9005915}
[2018-04-11 21:52:39] Started run 721 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:40] Error in run 714: Run already exists in server. Run id(s): {9005916}
[2018-04-11 21:52:40] Started run 722 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:41] Error in run 715: Run already exists in server. Run id(s): {9005921}
[2018-04-11 21:52:41] Started run 723 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:42] Error in run 716: Run already exists in server. Run id(s): {9005924}
[2018-04-11 21:52:42] Error in run 717: Run already exists in server. Run id(s): {9005922}
[2018-04-11 21:52:42] Error in run 718: Run already exists in server. Run id(s): {9005925}
[2018-04-11 21:52:42] Started run 724 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:42] Started run 725 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:42] Started run 726 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:42] Error in run 720: Run already exists in server. Run id(s): {9005927}
[2018-04-11 21:52:42] Error in run 719: Run already exists in server. Run id(s): {9005926}
[2018-04-11 21:52:42] Started run 727 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:42] Started run 728 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:44] Error in run 721: Run already exists in server. Run id(s): {9005930}
[2018-04-11 21:52:44] Started run 729 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:45] Error in run 722: Run already exists in server. Run id(s): {9005931}
[2018-04-11 21:52:45] Started run 730 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:45] Error in run 723: Run already exists in server. Run id(s): {9005938}
[2018-04-11 21:52:45] Started run 731 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:46] Error in run 724: Run already exists in server. Run id(s): {9005935}
[2018-04-11 21:52:46] Error in run 726: Run already exists in server. Run id(s): {9005944}
[2018-04-11 21:52:46] Started run 732 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:46] Error in run 725: Run already exists in server. Run id(s): {9005948}
[2018-04-11 21:52:46] Started run 733 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:46] Started run 734 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:47] Error in run 727: Run already exists in server. Run id(s): {9005937}
[2018-04-11 21:52:47] Started run 735 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:47] Error in run 728: Run already exists in server. Run id(s): {9005946}
[2018-04-11 21:52:47] Started run 736 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:49] Error in run 729: Run already exists in server. Run id(s): {9005952}
[2018-04-11 21:52:49] Started run 737 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:49] Error in run 730: Run already exists in server. Run id(s): {9005950}
[2018-04-11 21:52:49] Started run 738 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:50] Error in run 731: Run already exists in server. Run id(s): {9005954}
[2018-04-11 21:52:50] Started run 739 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:51] Error in run 732: Run already exists in server. Run id(s): {9005959}
[2018-04-11 21:52:51] Started run 740 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:51] Error in run 733: Run already exists in server. Run id(s): {9005957}
[2018-04-11 21:52:51] Started run 741 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:51] Error in run 734: Run already exists in server. Run id(s): {9005962}
[2018-04-11 21:52:51] Started run 742 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:51] Error in run 735: Run already exists in server. Run id(s): {9005963}
[2018-04-11 21:52:51] Started run 743 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:52] Error in run 736: Run already exists in server. Run id(s): {9005973}
[2018-04-11 21:52:52] Started run 744 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:53] Error in run 737: Run already exists in server. Run id(s): {9005974}
[2018-04-11 21:52:53] Started run 745 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:54] Error in run 738: Run already exists in server. Run id(s): {9005969}
[2018-04-11 21:52:54] Started run 746 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:54] Error in run 739: Run already exists in server. Run id(s): {9005972}
[2018-04-11 21:52:54] Started run 747 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:55] Error in run 740: Run already exists in server. Run id(s): {9005971}
[2018-04-11 21:52:55] Started run 748 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:55] Error in run 741: Run already exists in server. Run id(s): {9005978}
[2018-04-11 21:52:55] Started run 749 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:55] Error in run 742: Run already exists in server. Run id(s): {9005976}
[2018-04-11 21:52:55] Started run 750 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:56] Error in run 743: Run already exists in server. Run id(s): {9005982}
[2018-04-11 21:52:56] Started run 751 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:56] Error in run 744: Run already exists in server. Run id(s): {9005993}
[2018-04-11 21:52:56] Started run 752 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:58] Error in run 745: Run already exists in server. Run id(s): {9005987}
[2018-04-11 21:52:58] Started run 753 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:58] Error in run 746: Run already exists in server. Run id(s): {9005989}
[2018-04-11 21:52:58] Started run 754 on task 3913, dataset 'kc2'.
[2018-04-11 21:52:59] Error in run 747: Run already exists in server. Run id(s): {9005990}
[2018-04-11 21:52:59] Started run 755 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:00] Error in run 748: Run already exists in server. Run id(s): {9005991}
[2018-04-11 21:53:00] Started run 756 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:00] Error in run 749: Run already exists in server. Run id(s): {9005995}
[2018-04-11 21:53:00] Started run 757 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:00] Error in run 750: Run already exists in server. Run id(s): {9005998}
[2018-04-11 21:53:00] Started run 758 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:01] Error in run 751: Run already exists in server. Run id(s): {9005997}
[2018-04-11 21:53:01] Started run 759 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:01] Error in run 752: Run already exists in server. Run id(s): {9005999}
[2018-04-11 21:53:01] Started run 760 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:02] Error in run 753: Run already exists in server. Run id(s): {9006003}
[2018-04-11 21:53:02] Started run 761 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:03] Error in run 754: Run already exists in server. Run id(s): {9006000}
[2018-04-11 21:53:03] Started run 762 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:04] Error in run 755: Run already exists in server. Run id(s): {9006004}
[2018-04-11 21:53:04] Started run 763 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:05] Error in run 757: Run already exists in server. Run id(s): {9006010}
[2018-04-11 21:53:05] Error in run 758: Run already exists in server. Run id(s): {9006013}
[2018-04-11 21:53:05] Started run 764 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:05] Error in run 756: Run already exists in server. Run id(s): {9006009}
[2018-04-11 21:53:05] Started run 765 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:05] Started run 766 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:05] Error in run 759: Run already exists in server. Run id(s): {9006015}
[2018-04-11 21:53:05] Error in run 760: Run already exists in server. Run id(s): {9006018}
[2018-04-11 21:53:05] Started run 768 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:05] Started run 767 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:07] Error in run 761: Run already exists in server. Run id(s): {9006014}
[2018-04-11 21:53:07] Started run 769 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:08] Error in run 762: Run already exists in server. Run id(s): {9006023}
[2018-04-11 21:53:08] Started run 770 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:09] Error in run 763: Run already exists in server. Run id(s): {9006019}
[2018-04-11 21:53:09] Started run 771 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:09] Error in run 764: Run already exists in server. Run id(s): {9006021}
[2018-04-11 21:53:09] Started run 772 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:09] Error in run 766: Run already exists in server. Run id(s): {9006035}
[2018-04-11 21:53:09] Error in run 765: Run already exists in server. Run id(s): {9006025}
[2018-04-11 21:53:09] Started run 773 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:09] Started run 774 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:10] Error in run 767: Run already exists in server. Run id(s): {9006033}
[2018-04-11 21:53:10] Error in run 768: Run already exists in server. Run id(s): {9006031}
[2018-04-11 21:53:10] Started run 775 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:10] Started run 776 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:11] Error in run 769: Run already exists in server. Run id(s): {9006034}
[2018-04-11 21:53:11] Started run 777 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:12] Error in run 770: Run already exists in server. Run id(s): {9006040}
[2018-04-11 21:53:12] Started run 778 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:13] Error in run 771: Run already exists in server. Run id(s): {9006038}
[2018-04-11 21:53:13] Started run 779 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:14] Error in run 772: Run already exists in server. Run id(s): {9006043}
[2018-04-11 21:53:14] Started run 780 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:14] Error in run 774: Run already exists in server. Run id(s): {9006050}
[2018-04-11 21:53:14] Started run 781 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:14] Error in run 773: Run already exists in server. Run id(s): {9006044}
[2018-04-11 21:53:14] Started run 782 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:14] Error in run 775: Run already exists in server. Run id(s): {9006051}
[2018-04-11 21:53:14] Started run 783 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:14] Error in run 776: Run already exists in server. Run id(s): {9006052}
[2018-04-11 21:53:14] Started run 784 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:16] Error in run 777: Run already exists in server. Run id(s): {9006053}
[2018-04-11 21:53:16] Started run 785 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:17] Error in run 778: Run already exists in server. Run id(s): {9006054}
[2018-04-11 21:53:17] Started run 786 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:18] Error in run 779: Run already exists in server. Run id(s): {9006055}
[2018-04-11 21:53:18] Started run 787 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:18] Error in run 780: Run already exists in server. Run id(s): {9006057}
[2018-04-11 21:53:18] Started run 788 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:19] Error in run 782: Run already exists in server. Run id(s): {9006060}
[2018-04-11 21:53:19] Error in run 781: Run already exists in server. Run id(s): {9006058}
[2018-04-11 21:53:19] Started run 789 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:19] Started run 790 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:19] Error in run 783: Run already exists in server. Run id(s): {9006063}
[2018-04-11 21:53:19] Error in run 784: Run already exists in server. Run id(s): {9006061}
[2018-04-11 21:53:19] Started run 791 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:19] Started run 792 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:21] Error in run 785: Run already exists in server. Run id(s): {9006065}
[2018-04-11 21:53:21] Started run 793 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:21] Error in run 786: Run already exists in server. Run id(s): {9006064}
[2018-04-11 21:53:21] Started run 794 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:22] Error in run 787: Run already exists in server. Run id(s): {9006068}
[2018-04-11 21:53:22] Started run 795 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:23] Error in run 788: Run already exists in server. Run id(s): {9006070}
[2018-04-11 21:53:23] Started run 796 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:23] Error in run 790: Run already exists in server. Run id(s): {9006078}
[2018-04-11 21:53:23] Error in run 789: Run already exists in server. Run id(s): {9006079}
[2018-04-11 21:53:23] Started run 797 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:23] Started run 798 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:24] Error in run 792: Run already exists in server. Run id(s): {9006087}
[2018-04-11 21:53:24] Error in run 791: Run already exists in server. Run id(s): {9006080}
[2018-04-11 21:53:24] Started run 799 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:24] Started run 800 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:25] Error in run 793: Run already exists in server. Run id(s): {9006088}
[2018-04-11 21:53:25] Started run 801 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:26] Error in run 794: Run already exists in server. Run id(s): {9006086}
[2018-04-11 21:53:26] Started run 802 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:27] Error in run 795: Run already exists in server. Run id(s): {9006091}
[2018-04-11 21:53:27] Started run 803 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:28] Error in run 796: Run already exists in server. Run id(s): {9006092}
[2018-04-11 21:53:28] Started run 804 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:28] Error in run 797: Run already exists in server. Run id(s): {9006103}
[2018-04-11 21:53:28] Started run 805 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:28] Error in run 798: Run already exists in server. Run id(s): {9006096}
[2018-04-11 21:53:28] Started run 806 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:28] Error in run 799: Run already exists in server. Run id(s): {9006098}
[2018-04-11 21:53:28] Started run 807 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:28] Error in run 800: Run already exists in server. Run id(s): {9006101}
[2018-04-11 21:53:28] Started run 808 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:29] Error in run 801: Run already exists in server. Run id(s): {9006111}
[2018-04-11 21:53:29] Started run 809 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:31] Error in run 802: Run already exists in server. Run id(s): {9006102}
[2018-04-11 21:53:31] Started run 810 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:32] Error in run 803: Run already exists in server. Run id(s): {9006105}
[2018-04-11 21:53:32] Started run 811 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:32] Error in run 804: Run already exists in server. Run id(s): {9006108}
[2018-04-11 21:53:32] Error in run 805: Run already exists in server. Run id(s): {9006112}
[2018-04-11 21:53:32] Error in run 806: Run already exists in server. Run id(s): {9006110}
[2018-04-11 21:53:32] Started run 812 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:32] Started run 813 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:32] Started run 814 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:33] Error in run 807: Run already exists in server. Run id(s): {9006118}
[2018-04-11 21:53:33] Error in run 808: Run already exists in server. Run id(s): {9006117}
[2018-04-11 21:53:33] Started run 815 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:33] Started run 816 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:34] Error in run 809: Run already exists in server. Run id(s): {9006115}
[2018-04-11 21:53:34] Started run 817 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:36] Error in run 810: Run already exists in server. Run id(s): {9006123}
[2018-04-11 21:53:36] Started run 818 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:36] Error in run 811: Run already exists in server. Run id(s): {9006128}
[2018-04-11 21:53:36] Started run 819 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:37] Error in run 812: Run already exists in server. Run id(s): {9006120}
[2018-04-11 21:53:37] Started run 820 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:37] Error in run 814: Run already exists in server. Run id(s): {8969894}
[2018-04-11 21:53:37] Error in run 813: Run already exists in server. Run id(s): {9006125}
[2018-04-11 21:53:37] Started run 821 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:37] Started run 822 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:38] Error in run 815: Run already exists in server. Run id(s): {9006130}
[2018-04-11 21:53:38] Started run 823 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:38] Error in run 816: Run already exists in server. Run id(s): {9006129}
[2018-04-11 21:53:38] Started run 824 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:39] Error in run 817: Run already exists in server. Run id(s): {9006135}
[2018-04-11 21:53:39] Started run 825 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:40] Error in run 818: Run already exists in server. Run id(s): {9006131}
[2018-04-11 21:53:40] Started run 826 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:41] Error in run 819: Run already exists in server. Run id(s): {8969917}
[2018-04-11 21:53:41] Started run 827 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:42] Error in run 820: Run already exists in server. Run id(s): {9006142}
[2018-04-11 21:53:42] Started run 828 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:42] Error in run 822: Run already exists in server. Run id(s): {9006139}
[2018-04-11 21:53:42] Started run 829 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:42] Error in run 821: Run already exists in server. Run id(s): {9006137}
[2018-04-11 21:53:42] Started run 830 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:43] Error in run 823: Run already exists in server. Run id(s): {9006151}
[2018-04-11 21:53:43] Started run 831 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:43] Error in run 824: Run already exists in server. Run id(s): {9006141}
[2018-04-11 21:53:43] Started run 832 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:44] Error in run 825: Run already exists in server. Run id(s): {9006145}
[2018-04-11 21:53:44] Started run 833 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:45] Error in run 826: Run already exists in server. Run id(s): {9006144}
[2018-04-11 21:53:45] Started run 834 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:45] Error in run 827: Run already exists in server. Run id(s): {9006152}
[2018-04-11 21:53:45] Started run 835 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:46] Error in run 828: Run already exists in server. Run id(s): {9006157}
[2018-04-11 21:53:46] Started run 836 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:47] Error in run 829: Run already exists in server. Run id(s): {8969895}
[2018-04-11 21:53:47] Started run 837 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:47] Error in run 830: Run already exists in server. Run id(s): {9006163}
[2018-04-11 21:53:47] Started run 838 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:47] Error in run 832: Run already exists in server. Run id(s): {9006167}
[2018-04-11 21:53:47] Error in run 831: Run already exists in server. Run id(s): {9006165}
[2018-04-11 21:53:47] Started run 839 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:47] Started run 840 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:48] Error in run 833: Run already exists in server. Run id(s): {8969918}
[2018-04-11 21:53:48] Started run 841 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:50] Error in run 834: Run already exists in server. Run id(s): {8969897}
[2018-04-11 21:53:50] Started run 842 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:50] Error in run 835: Run already exists in server. Run id(s): {9006176}
[2018-04-11 21:53:50] Started run 843 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:51] Error in run 836: Run already exists in server. Run id(s): {9006164}
[2018-04-11 21:53:51] Started run 844 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:51] Error in run 838: Run already exists in server. Run id(s): {9006174}
[2018-04-11 21:53:51] Started run 845 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:52] Error in run 837: Run already exists in server. Run id(s): {8969906}
[2018-04-11 21:53:52] Started run 846 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:52] Error in run 840: Run already exists in server. Run id(s): {8969923}
[2018-04-11 21:53:52] Error in run 839: Run already exists in server. Run id(s): {9006173}
[2018-04-11 21:53:52] Started run 847 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:52] Started run 848 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:53] Error in run 841: Run already exists in server. Run id(s): {9006177}
[2018-04-11 21:53:53] Started run 849 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:55] Error in run 843: Run already exists in server. Run id(s): {9006180}
[2018-04-11 21:53:55] Started run 850 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:55] Error in run 842: Run already exists in server. Run id(s): {9006178}
[2018-04-11 21:53:55] Started run 851 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:55] Error in run 844: Run already exists in server. Run id(s): {9006181}
[2018-04-11 21:53:55] Started run 852 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:56] Error in run 845: Run already exists in server. Run id(s): {8969931}
[2018-04-11 21:53:56] Error in run 846: Run already exists in server. Run id(s): {9006192}
[2018-04-11 21:53:56] Started run 853 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:56] Started run 854 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:57] Error in run 847: Run already exists in server. Run id(s): {8969919}
[2018-04-11 21:53:57] Started run 855 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:57] Error in run 848: Run already exists in server. Run id(s): {9006185}
[2018-04-11 21:53:57] Started run 856 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:58] Error in run 849: Run already exists in server. Run id(s): {9006186}
[2018-04-11 21:53:58] Started run 857 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:59] Error in run 851: Run already exists in server. Run id(s): {9006196}
[2018-04-11 21:53:59] Started run 858 on task 3913, dataset 'kc2'.
[2018-04-11 21:53:59] Error in run 850: Run already exists in server. Run id(s): {8969920, 9019851}
[2018-04-11 21:53:59] Started run 859 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:00] Error in run 852: Run already exists in server. Run id(s): {8969921}
[2018-04-11 21:54:00] Started run 860 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:01] Error in run 854: Run already exists in server. Run id(s): {9006198}
[2018-04-11 21:54:01] Error in run 853: Run already exists in server. Run id(s): {9006190}
[2018-04-11 21:54:01] Started run 861 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:01] Started run 862 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:02] Error in run 855: Run already exists in server. Run id(s): {8969899}
[2018-04-11 21:54:02] Started run 863 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:02] Error in run 856: Run already exists in server. Run id(s): {9006204}
[2018-04-11 21:54:02] Started run 864 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:02] Error in run 857: Run already exists in server. Run id(s): {8969908}
[2018-04-11 21:54:03] Started run 865 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:04] Error in run 858: Run already exists in server. Run id(s): {9006205}
[2018-04-11 21:54:04] Error in run 859: Run already exists in server. Run id(s): {9006209}
[2018-04-11 21:54:04] Started run 866 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:04] Started run 867 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:05] Error in run 860: Run already exists in server. Run id(s): {9006208}
[2018-04-11 21:54:05] Started run 868 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:06] Error in run 862: Run already exists in server. Run id(s): {9006213}
[2018-04-11 21:54:06] Error in run 861: Run already exists in server. Run id(s): {8969896}
[2018-04-11 21:54:06] Started run 869 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:06] Started run 870 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:06] Error in run 863: Run already exists in server. Run id(s): {9006221}
[2018-04-11 21:54:06] Started run 871 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:06] Error in run 864: Run already exists in server. Run id(s): {9006215}
[2018-04-11 21:54:06] Started run 872 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:07] Error in run 865: Run already exists in server. Run id(s): {9006218}
[2018-04-11 21:54:07] Started run 873 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:09] Error in run 867: Run already exists in server. Run id(s): {9006217}
[2018-04-11 21:54:09] Error in run 866: Run already exists in server. Run id(s): {8969909}
[2018-04-11 21:54:09] Started run 874 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:09] Started run 875 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:09] Error in run 868: Run already exists in server. Run id(s): {8969898}
[2018-04-11 21:54:09] Started run 876 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:11] Error in run 870: Run already exists in server. Run id(s): {9006222}
[2018-04-11 21:54:11] Error in run 869: Run already exists in server. Run id(s): {8969912}
[2018-04-11 21:54:11] Started run 877 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:11] Started run 878 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:11] Error in run 871: Run already exists in server. Run id(s): {8969902}
[2018-04-11 21:54:11] Started run 879 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:11] Error in run 872: Run already exists in server. Run id(s): {9006238}
[2018-04-11 21:54:11] Started run 880 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:12] Error in run 873: Run already exists in server. Run id(s): {8969926}
[2018-04-11 21:54:12] Started run 881 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:14] Error in run 874: Run already exists in server. Run id(s): {9006228}
[2018-04-11 21:54:14] Started run 882 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:14] Error in run 875: Run already exists in server. Run id(s): {8969913}
[2018-04-11 21:54:14] Started run 883 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:14] Error in run 876: Run already exists in server. Run id(s): {9006236}
[2018-04-11 21:54:14] Started run 884 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:15] Error in run 877: Run already exists in server. Run id(s): {8969910}
[2018-04-11 21:54:15] Started run 885 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:15] Error in run 878: Run already exists in server. Run id(s): {8969911}
[2018-04-11 21:54:15] Started run 886 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:15] Error in run 879: Run already exists in server. Run id(s): {9006237}
[2018-04-11 21:54:15] Started run 887 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:16] Error in run 880: Run already exists in server. Run id(s): {9006240}
[2018-04-11 21:54:16] Started run 888 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:17] Error in run 881: Run already exists in server. Run id(s): {9006244}
[2018-04-11 21:54:17] Started run 889 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:18] Error in run 882: Run already exists in server. Run id(s): {8969916}
[2018-04-11 21:54:18] Started run 890 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:18] Error in run 884: Run already exists in server. Run id(s): {8969905}
[2018-04-11 21:54:18] Started run 891 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:18] Error in run 883: Run already exists in server. Run id(s): {8969901}
[2018-04-11 21:54:18] Started run 892 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:20] Error in run 885: Run already exists in server. Run id(s): {8969914}
[2018-04-11 21:54:20] Started run 893 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:20] Error in run 887: Run already exists in server. Run id(s): {8969922}
[2018-04-11 21:54:20] Error in run 886: Run already exists in server. Run id(s): {9006243}
[2018-04-11 21:54:20] Started run 894 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:20] Started run 895 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:21] Error in run 888: Run already exists in server. Run id(s): {8969907}
[2018-04-11 21:54:21] Started run 896 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:22] Error in run 889: Run already exists in server. Run id(s): {9006248}
[2018-04-11 21:54:22] Started run 897 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:22] Error in run 890: Run already exists in server. Run id(s): {8969904}
[2018-04-11 21:54:22] Started run 898 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:23] Error in run 891: Run already exists in server. Run id(s): {9006250}
[2018-04-11 21:54:23] Error in run 892: Run already exists in server. Run id(s): {9006249}
[2018-04-11 21:54:23] Started run 899 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:23] Started run 900 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:24] Error in run 893: Run already exists in server. Run id(s): {9006252}
[2018-04-11 21:54:24] Started run 901 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:25] Error in run 895: Run already exists in server. Run id(s): {9006258}
[2018-04-11 21:54:25] Started run 902 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:25] Error in run 894: Run already exists in server. Run id(s): {9006262}
[2018-04-11 21:54:25] Started run 903 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:26] Error in run 896: Run already exists in server. Run id(s): {9006267}
[2018-04-11 21:54:26] Started run 904 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:26] Error in run 897: Run already exists in server. Run id(s): {9006266}
[2018-04-11 21:54:26] Started run 905 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:27] Error in run 898: Run already exists in server. Run id(s): {8969928}
[2018-04-11 21:54:27] Started run 906 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:28] Error in run 899: Run already exists in server. Run id(s): {9006269}
[2018-04-11 21:54:28] Started run 907 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:28] Error in run 900: Run already exists in server. Run id(s): {8969927}
[2018-04-11 21:54:28] Started run 908 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:29] Error in run 901: Run already exists in server. Run id(s): {8969924}
[2018-04-11 21:54:29] Started run 909 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:30] Error in run 903: Run already exists in server. Run id(s): {9006273}
[2018-04-11 21:54:30] Error in run 902: Run already exists in server. Run id(s): {9006272}
[2018-04-11 21:54:30] Started run 910 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:30] Started run 911 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:30] Error in run 904: Run already exists in server. Run id(s): {9006271}
[2018-04-11 21:54:30] Started run 912 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:31] Error in run 905: Run already exists in server. Run id(s): {8969925}
[2018-04-11 21:54:31] Started run 913 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:32] Error in run 906: Run already exists in server. Run id(s): {9006275}
[2018-04-11 21:54:32] Started run 914 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:33] Error in run 908: Run already exists in server. Run id(s): {9006287}
[2018-04-11 21:54:33] Error in run 907: Run already exists in server. Run id(s): {9006281}
[2018-04-11 21:54:33] Started run 916 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:33] Started run 915 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:34] Error in run 909: Run already exists in server. Run id(s): {8969936}
[2018-04-11 21:54:34] Started run 917 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:34] Error in run 910: Run already exists in server. Run id(s): {9006286}
[2018-04-11 21:54:34] Started run 918 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:34] Error in run 911: Run already exists in server. Run id(s): {9006285}
[2018-04-11 21:54:34] Started run 919 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:35] Error in run 912: Run already exists in server. Run id(s): {9006288}
[2018-04-11 21:54:35] Started run 920 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:36] Error in run 913: Run already exists in server. Run id(s): {9006296}
[2018-04-11 21:54:36] Started run 921 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:37] Error in run 914: Run already exists in server. Run id(s): {9006294}
[2018-04-11 21:54:37] Started run 922 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:37] Error in run 915: Run already exists in server. Run id(s): {9006292}
[2018-04-11 21:54:37] Started run 923 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:37] Error in run 916: Run already exists in server. Run id(s): {8969929}
[2018-04-11 21:54:37] Started run 924 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:38] Error in run 917: Run already exists in server. Run id(s): {9006302}
[2018-04-11 21:54:38] Started run 925 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:39] Error in run 919: Run already exists in server. Run id(s): {9006305}
[2018-04-11 21:54:39] Started run 926 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:39] Error in run 918: Run already exists in server. Run id(s): {9006300}
[2018-04-11 21:54:39] Started run 927 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:40] Error in run 920: Run already exists in server. Run id(s): {9006307}
[2018-04-11 21:54:40] Started run 928 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:41] Error in run 921: Run already exists in server. Run id(s): {9006306}
[2018-04-11 21:54:41] Started run 929 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:41] Error in run 922: Run already exists in server. Run id(s): {9006308}
[2018-04-11 21:54:41] Started run 930 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:42] Error in run 923: Run already exists in server. Run id(s): {9006312}
[2018-04-11 21:54:42] Started run 931 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:42] Error in run 924: Run already exists in server. Run id(s): {9006313}
[2018-04-11 21:54:42] Started run 932 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:43] Error in run 925: Run already exists in server. Run id(s): {8969933}
[2018-04-11 21:54:43] Started run 933 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:44] Error in run 927: Run already exists in server. Run id(s): {9006325}
[2018-04-11 21:54:44] Started run 934 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:44] Error in run 926: Run already exists in server. Run id(s): {9006322}
[2018-04-11 21:54:44] Started run 935 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:44] Error in run 928: Run already exists in server. Run id(s): {8969932}
[2018-04-11 21:54:44] Started run 936 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:45] Error in run 929: Run already exists in server. Run id(s): {9006324}
[2018-04-11 21:54:45] Started run 937 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:46] Error in run 930: Run already exists in server. Run id(s): {8969935}
[2018-04-11 21:54:46] Started run 938 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:46] Error in run 931: Run already exists in server. Run id(s): {8969934}
[2018-04-11 21:54:46] Started run 939 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:47] Error in run 932: Run already exists in server. Run id(s): {9006329}
[2018-04-11 21:54:47] Started run 940 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:48] Error in run 933: Run already exists in server. Run id(s): {9006340}
[2018-04-11 21:54:48] Started run 941 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:49] Error in run 935: Run already exists in server. Run id(s): {9006336}
[2018-04-11 21:54:49] Error in run 934: Run already exists in server. Run id(s): {9006327}
[2018-04-11 21:54:49] Started run 942 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:49] Started run 943 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:49] Error in run 936: Run already exists in server. Run id(s): {9006342}
[2018-04-11 21:54:49] Started run 944 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:50] Error in run 937: Run already exists in server. Run id(s): {9006343}
[2018-04-11 21:54:50] Started run 945 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:51] Error in run 938: Run already exists in server. Run id(s): {9006345}
[2018-04-11 21:54:51] Started run 946 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:51] Error in run 939: Run already exists in server. Run id(s): {9006339}
[2018-04-11 21:54:51] Started run 947 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:52] Error in run 940: Run already exists in server. Run id(s): {9006346}
[2018-04-11 21:54:52] Started run 948 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:53] Error in run 941: Run already exists in server. Run id(s): {9006349}
[2018-04-11 21:54:53] Started run 949 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:53] Error in run 942: Run already exists in server. Run id(s): {9006353}
[2018-04-11 21:54:53] Started run 950 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:53] Error in run 943: Run already exists in server. Run id(s): {9006355}
[2018-04-11 21:54:53] Started run 951 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:54] Error in run 944: Run already exists in server. Run id(s): {9006357}
[2018-04-11 21:54:54] Started run 952 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:54] Error in run 945: Run already exists in server. Run id(s): {9006354}
[2018-04-11 21:54:54] Started run 953 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:55] Error in run 947: Run already exists in server. Run id(s): {9006360}
[2018-04-11 21:54:55] Started run 954 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:56] Error in run 946: Run already exists in server. Run id(s): {9006359}
[2018-04-11 21:54:56] Started run 955 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:56] Error in run 948: Run already exists in server. Run id(s): {9006361}
[2018-04-11 21:54:56] Started run 956 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:57] Error in run 949: Run already exists in server. Run id(s): {9006364}
[2018-04-11 21:54:57] Started run 957 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:58] Error in run 950: Run already exists in server. Run id(s): {9006366}
[2018-04-11 21:54:58] Started run 958 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:58] Error in run 951: Run already exists in server. Run id(s): {9006370}
[2018-04-11 21:54:58] Started run 959 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:59] Error in run 952: Run already exists in server. Run id(s): {9006372}
[2018-04-11 21:54:59] Started run 960 on task 3913, dataset 'kc2'.
[2018-04-11 21:54:59] Error in run 953: Run already exists in server. Run id(s): {9006376}
[2018-04-11 21:54:59] Started run 961 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:00] Error in run 954: Run already exists in server. Run id(s): {9006382}
[2018-04-11 21:55:00] Started run 962 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:00] Error in run 955: Run already exists in server. Run id(s): {9006379}
[2018-04-11 21:55:00] Started run 963 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:01] Error in run 956: Run already exists in server. Run id(s): {9006387}
[2018-04-11 21:55:01] Started run 964 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:02] Error in run 957: Run already exists in server. Run id(s): {9006385}
[2018-04-11 21:55:02] Started run 965 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:03] Error in run 958: Run already exists in server. Run id(s): {9006383}
[2018-04-11 21:55:03] Started run 966 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:03] Error in run 959: Run already exists in server. Run id(s): {9006388}
[2018-04-11 21:55:03] Started run 967 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:03] Error in run 960: Run already exists in server. Run id(s): {9006395}
[2018-04-11 21:55:03] Started run 968 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:03] Error in run 961: Run already exists in server. Run id(s): {9006392}
[2018-04-11 21:55:03] Started run 969 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:05] Error in run 962: Run already exists in server. Run id(s): {9006393}
[2018-04-11 21:55:05] Started run 970 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:05] Error in run 963: Run already exists in server. Run id(s): {9006403}
[2018-04-11 21:55:05] Started run 971 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:05] Error in run 964: Run already exists in server. Run id(s): {9006398}
[2018-04-11 21:55:05] Started run 972 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:06] Error in run 965: Run already exists in server. Run id(s): {9006406}
[2018-04-11 21:55:06] Started run 973 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:07] Error in run 966: Run already exists in server. Run id(s): {9006404}
[2018-04-11 21:55:07] Started run 974 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:07] Error in run 967: Run already exists in server. Run id(s): {9006400}
[2018-04-11 21:55:07] Started run 975 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:08] Error in run 968: Run already exists in server. Run id(s): {9006408}
[2018-04-11 21:55:08] Started run 976 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:08] Error in run 969: Run already exists in server. Run id(s): {9006409}
[2018-04-11 21:55:08] Started run 977 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:09] Error in run 970: Run already exists in server. Run id(s): {9006418}
[2018-04-11 21:55:09] Started run 978 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:10] Error in run 971: Run already exists in server. Run id(s): {9006419}
[2018-04-11 21:55:10] Started run 979 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:10] Error in run 972: Run already exists in server. Run id(s): {9006423}
[2018-04-11 21:55:10] Started run 980 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:11] Error in run 973: Run already exists in server. Run id(s): {9006422}
[2018-04-11 21:55:11] Started run 981 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:12] Error in run 974: Run already exists in server. Run id(s): {9006426}
[2018-04-11 21:55:12] Started run 982 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:12] Error in run 975: Run already exists in server. Run id(s): {9006425}
[2018-04-11 21:55:12] Started run 983 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:12] Error in run 976: Run already exists in server. Run id(s): {9006430}
[2018-04-11 21:55:12] Error in run 977: Run already exists in server. Run id(s): {9006428}
[2018-04-11 21:55:12] Started run 984 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:12] Started run 985 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:14] Error in run 978: Run already exists in server. Run id(s): {9006433}
[2018-04-11 21:55:14] Started run 986 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:14] Error in run 979: Run already exists in server. Run id(s): {9006434}
[2018-04-11 21:55:14] Started run 987 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:15] Error in run 980: Run already exists in server. Run id(s): {9006440}
[2018-04-11 21:55:15] Started run 988 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:15] Error in run 981: Run already exists in server. Run id(s): {9006436}
[2018-04-11 21:55:15] Started run 989 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:16] Error in run 982: Run already exists in server. Run id(s): {9006437}
[2018-04-11 21:55:16] Started run 990 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:16] Error in run 983: Run already exists in server. Run id(s): {9006438}
[2018-04-11 21:55:16] Started run 991 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:17] Error in run 985: Run already exists in server. Run id(s): {9006441}
[2018-04-11 21:55:17] Started run 992 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:17] Error in run 984: Run already exists in server. Run id(s): {9006442}
[2018-04-11 21:55:17] Started run 993 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:18] Error in run 986: Run already exists in server. Run id(s): {9006447}
[2018-04-11 21:55:18] Started run 994 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:19] Error in run 987: Run already exists in server. Run id(s): {9006448}
[2018-04-11 21:55:19] Started run 995 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:19] Error in run 988: Run already exists in server. Run id(s): {9006452}
[2018-04-11 21:55:19] Started run 996 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:20] Error in run 989: Run already exists in server. Run id(s): {9006451}
[2018-04-11 21:55:20] Started run 997 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:21] Error in run 990: Run already exists in server. Run id(s): {9006456}
[2018-04-11 21:55:21] Started run 998 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:21] Error in run 991: Run already exists in server. Run id(s): {9006453}
[2018-04-11 21:55:21] Started run 999 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:22] Error in run 992: Run already exists in server. Run id(s): {9006450}
[2018-04-11 21:55:22] Error in run 993: Run already exists in server. Run id(s): {9006454}
[2018-04-11 21:55:22] Started run 1000 on task 3913, dataset 'kc2'.
[2018-04-11 21:55:23] Error in run 994: Run already exists in server. Run id(s): {9006464}
[2018-04-11 21:55:23] Error in run 995: Run already exists in server. Run id(s): {9006465}
[2018-04-11 21:55:24] Error in run 996: Run already exists in server. Run id(s): {9006470}
[2018-04-11 21:55:24] Error in run 997: Run already exists in server. Run id(s): {9006469}
[2018-04-11 21:55:25] Error in run 998: Run already exists in server. Run id(s): {9006471}
[2018-04-11 21:55:26] Error in run 999: Run already exists in server. Run id(s): {9006475}
[2018-04-11 21:55:27] Error in run 1000: Run already exists in server. Run id(s): {9006472}
